diff --git a/PKGBUILD b/PKGBUILD
index 3ccf5a8..b66b114 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,8 +2,8 @@
 # Contributor: Wyatt J. Brown <sushidudeteam at gmail dot com>
 # shellcheck disable=SC2034,SC2148,SC2154
 pkgname=all-repository-fonts
-pkgver=20231210
-pkgrel=2
+pkgver=20240402
+pkgrel=1
 pkgdesc='Meta package for all fonts in the official repositories.'
 arch=('any')
 url='https://aur.archlinux.org/packages/all-repository-fonts/'
@@ -16,6 +16,7 @@ depends=(
 	'adobe-source-han-sans-otc-fonts'
 	'adobe-source-han-sans-tw-fonts'
 	'adobe-source-han-serif-cn-fonts'
+	'adobe-source-han-serif-hk-fonts'
 	'adobe-source-han-serif-jp-fonts'
 	'adobe-source-han-serif-kr-fonts'
 	'adobe-source-han-serif-otc-fonts'
@@ -24,7 +25,6 @@ depends=(
 	'adobe-source-serif-fonts'
 	'awesome-terminal-fonts'
 	'cantarell-fonts'
-	'gentium-plus-font'
 	'gnu-free-fonts'
 	'gsfonts'
 	'inter-font'
@@ -34,12 +34,15 @@ depends=(
 	'noto-fonts-emoji'
 	'noto-fonts-extra'
 	'opendesktop-fonts'
+	'otf-atkinson-hyperlegible'
 	'otf-aurulent-nerd'
 	'otf-cascadia-code'
 	'otf-codenewroman-nerd'
 	'otf-comicshanns-nerd'
 	'otf-commit-mono-nerd'
 	'otf-cormorant'
+	'otf-crimson'
+	'otf-crimson-pro'
 	'otf-droid-nerd'
 	'otf-fantasque-sans-mono'
 	'otf-fira-mono'
@@ -48,13 +51,17 @@ depends=(
 	'otf-font-awesome'
 	'otf-geist-mono-nerd'
 	'otf-hasklig-nerd'
+	'otf-hermit'
 	'otf-hermit-nerd'
 	'otf-ipaexfont'
 	'otf-ipafont'
 	'otf-ipamjfont'
+	'otf-junicode'
 	'otf-latin-modern'
 	'otf-latinmodern-math'
+	'otf-monaspace'
 	'otf-monaspace-nerd'
+	'otf-montserrat'
 	'otf-opendyslexic-nerd'
 	'otf-overpass'
 	'otf-overpass-nerd'
@@ -124,6 +131,7 @@ depends=(
 	'ttf-fira-sans'
 	'ttf-firacode-nerd'
 	'ttf-font-awesome'
+	'ttf-gentium-plus'
 	'ttf-go-nerd'
 	'ttf-hack'
 	'ttf-hack-nerd'
@@ -157,21 +165,21 @@ depends=(
 	'ttf-linux-libertine-g'
 	'ttf-martian-mono-nerd'
 	'ttf-meslo-nerd'
+	'ttf-mona-sans'
 	'ttf-monaspace-variable'
 	'ttf-monofur'
 	'ttf-monofur-nerd'
 	'ttf-monoid'
 	'ttf-monoid-nerd'
 	'ttf-mononoki-nerd'
+	'ttf-montserrat'
 	'ttf-mplus-nerd'
 	'ttf-nerd-fonts-symbols'
-	'ttf-nerd-fonts-symbols-common'
 	'ttf-nerd-fonts-symbols-mono'
 	'ttf-noto-nerd'
 	'ttf-opensans'
 	'ttf-overpass'
 	'ttf-profont-nerd'
-	'ttf-proggy-clean'
 	'ttf-proggyclean-nerd'
 	'ttf-roboto'
 	'ttf-roboto-mono'
@@ -189,7 +197,6 @@ depends=(
 	'ttf-ubuntu-mono-nerd'
 	'ttf-ubuntu-nerd'
 	'ttf-victor-mono-nerd'
-	'xorg-fonts-type1'
 	'wqy-bitmapfont'
 	'wqy-microhei'
 	'wqy-microhei-lite'
@@ -198,4 +205,5 @@ depends=(
 	'xorg-fonts-75dpi'
 	'xorg-fonts-cyrillic'
 	'xorg-fonts-misc'
+	'xorg-fonts-type1'
 )
