diff --git a/PKGBUILD b/PKGBUILD
index 37ee7b4..e21f83d 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,7 +2,7 @@
 # Contributor: Wyatt J. Brown <sushidudeteam at gmail dot com>
 # shellcheck disable=SC2034,SC2148,SC2154
 pkgname=all-repository-fonts
-pkgver=20231101
+pkgver=20231108
 pkgrel=1
 pkgdesc='Meta package for all fonts in the official repositories.'
 arch=('any')
@@ -16,6 +16,7 @@ depends=(
 	'adobe-source-han-sans-otc-fonts'
 	'adobe-source-han-sans-tw-fonts'
 	'adobe-source-han-serif-cn-fonts'
+	'adobe-source-han-serif-hk-fonts'
 	'adobe-source-han-serif-jp-fonts'
 	'adobe-source-han-serif-kr-fonts'
 	'adobe-source-han-serif-otc-fonts'
@@ -34,12 +35,16 @@ depends=(
 	'noto-fonts-emoji'
 	'noto-fonts-extra'
 	'opendesktop-fonts'
+	'otf-atkinson-hyperlegible'
 	'otf-cascadia-code'
 	'otf-cormorant'
+	'otf-crimson'
+	'otf-crimson-pro'
 	'otf-fantasque-sans-mono'
 	'otf-fira-mono'
 	'otf-fira-sans'
 	'otf-font-awesome'
+	'otf-hermit'
 	'otf-ipaexfont'
 	'otf-ipafont'
 	'otf-ipamjfont'
@@ -76,12 +81,16 @@ depends=(
 	'ttf-anonymous-pro'
 	'ttf-arphic-ukai'
 	'ttf-arphic-uming'
+	'ttf-atkinson-hyperlegible'
 	'ttf-baekmuk'
 	'ttf-bitstream-vera'
 	'ttf-caladea'
 	'ttf-carlito'
 	'ttf-cascadia-code'
 	'ttf-cormorant'
+	'ttf-crimson'
+	'ttf-crimson-pro'
+	'ttf-crimson-pro-variable'
 	'ttf-croscore'
 	'ttf-dejavu'
 	'ttf-droid'
@@ -97,6 +106,7 @@ depends=(
 	'ttf-ibm-plex'
 	'ttf-inconsolata'
 	'ttf-indic-otf'
+	'ttf-input'
 	'ttf-jetbrains-mono'
 	'ttf-joypixels'
 	'ttf-junicode'
@@ -104,8 +114,8 @@ depends=(
 	'ttf-lato'
 	'ttf-liberation'
 	'ttf-monofur'
+	'ttf-monoid'
 	'ttf-nerd-fonts-symbols'
-	'ttf-nerd-fonts-symbols-common'
 	'ttf-nerd-fonts-symbols-mono'
 	'ttf-opensans'
 	'ttf-proggy-clean'
@@ -113,6 +123,7 @@ depends=(
 	'ttf-roboto-mono'
 	'ttf-sarasa-gothic'
 	'ttf-sazanami'
+	'ttf-scheherazade-new'
 	'ttf-tibetan-machine'
 	'ttf-ubuntu-font-family'
 	'wqy-bitmapfont'
