diff --git a/PKGBUILD b/PKGBUILD
index 70b40ca..2040b5e 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,12 +7,12 @@ pkgname=(
   'avidemux-qt6-git'
   'avidemux-cli-git'
 )
-pkgver=2.8.2.240125.cdbfa6956
+pkgver=2.8.2.r10039.ge652233d4
 pkgrel=1
 pkgdesc="A graphical/cli tool to edit video (filter/re-encode/split). (GIT version)"
 arch=('x86_64')
 url='http://www.avidemux.org'
-license=('GPL2')
+license=('GPL-2.0-or-later')
 makedepends=(
   'git'
   'cmake'
@@ -20,8 +20,6 @@ makedepends=(
   'nasm'
   'aom'
   'a52dec'
-  'aften'
-  'dcaenc'
   'faac'
   'faad2'
   'fribidi'
@@ -41,7 +39,9 @@ makedepends=(
   'twolame'
   'xvidcore'
   'x264'
+  'libx264.so'
   'x265'
+  'libx265.so'
   'sqlite'
   'glu'
   'qt5-script'
@@ -72,12 +72,10 @@ sha256sums=(
   '3c91ddefadfad5f43d514941ddfa1942ea2c618525aaae36bde5a5d18f8e2067'
 )
 
-options=('debug')
-
 pkgver() {
   cd avidemux
   _ver="$(cat cmake/avidemuxVersion.cmake | grep -m3 -e CPACK_PACKAGE_VERSION_MAJOR -e CPACK_PACKAGE_VERSION_MINOR -e CPACK_PACKAGE_VERSION_P | grep -o "[[:digit:]]*" | paste -sd'.')"
-  echo -e "${_ver}.$(date -u +%g%m%d).$(git rev-parse --short HEAD | head -c 11)"
+  printf '%s.r%s.g%s' "${_ver}" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
 }
 
 prepare() {
@@ -228,14 +226,12 @@ package_avidemux-core-git() {
     'a52dec' # 'liba52.so'
     'alsa-lib' 'libasound.so'
     'aom' 'libaom.so'
-    'aften' # 'libaften.so'
-    'dcaenc' # 'libdcaenc.so'
     'faac' 'libfaac.so'
     'faad2' 'libfaad.so'
     'gcc-libs' # 'libgcc_s.so' 'libstdc++.so'
     'glibc' # 'libc.so' 'libm.so'
     'fribidi'
-    'jack2' 'libjack.so'
+    'jack' 'libjack.so'
     'lame' 'libmp3lame.so'
     'libass' 'libass.so'
     'libfdk-aac' 'libfdk-aac.so'
@@ -286,7 +282,7 @@ package_avidemux-core-git() {
 package_avidemux-qt5-git() {
   pkgdesc="Qt5 GUI for Avidemux. (GIT version)"
   depends=(
-    "avidemux-core-git=${pkgver}"
+    "avidemux-core-git=${pkgver}-${pkgrel}"
     'gcc-libs' # 'libgcc_s.so' 'libstdc++.so'
     'glibc' # 'libc.so' 'libm.so'
     'glu' # 'libGLU.so'
@@ -326,7 +322,7 @@ package_avidemux-qt5-git() {
 package_avidemux-qt6-git() {
   pkgdesc="Qt6 GUI for Avidemux. (GIT version)"
   depends=(
-    "avidemux-core-git=${pkgver}"
+    "avidemux-core-git=${pkgver}-${pkgrel}"
     'gcc-libs' # 'libgcc_s.so' 'libstdc++.so'
     'glibc' # 'libc.so' 'libm.so'
     'glu' # 'libGLU.so'
@@ -368,7 +364,7 @@ package_avidemux-qt6-git() {
 package_avidemux-cli-git() {
   pkgdesc="CLI frontend for Avidemux. (GIT version)"
   depends=(
-    "avidemux-core-git=${pkgver}"
+    "avidemux-core-git=${pkgver}-${pkgrel}"
     'gcc-libs' # 'libgcc_s.so' 'libstdc++.so'
     'glibc' # 'libc.so' 'libm.so'
     )
