diff --git a/PKGBUILD b/PKGBUILD
index ce370db..bc5a9a4 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,13 +3,13 @@
 
 pkgname="ayatana-webmail"
 pkgver="23.10.13"
-pkgrel="1"
+pkgrel="2"
 pkgdesc="Webmail notifications and actions for any desktop"
 arch=("any")
 url="https://tari.in/www/software/ayatana-webmail/"
 license=("GPL3")
-depends=("hicolor-icon-theme" "dconf" "gobject-introspection" "desktop-file-utils" "libnotify" "libcanberra" "gnome-keyring" "python-babel" "python-gobject" "python-psutil" "python-secretstorage" "python-urllib3" "gsettings-desktop-schemas")
-makedepends=("python-setuptools" "python-polib")
+depends=("hicolor-icon-theme" "dconf" "gobject-introspection" "desktop-file-utils" "libnotify" "libcanberra" "gnome-keyring" "python-babel" "python-gobject" "python-psutil" "python-secretstorage" "python-urllib3" "gsettings-desktop-schemas" "python")
+makedepends=("python-build" "python-installer" "python-wheel" "python-setuptools" "python-polib")
 install="${pkgname}.install"
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/AyatanaIndicators/${pkgname}/archive/${pkgver}.tar.gz")
 md5sums=("3fccb9b6068095a735345ec6f8e725bc")
@@ -19,11 +19,11 @@ conflicts=("unity-mail")
 build()
 {
     cd ${pkgname}-${pkgver}
-    python setup.py build
+    python -m build -w -n
 }
 
 package()
 {
     cd ${pkgname}-${pkgver}
-    python setup.py install --root="${pkgdir}" --optimize=1
+    python -m installer -d "${pkgdir}" dist/*.whl
 }
