diff --git a/PKGBUILD b/PKGBUILD
index 6f574ea..03ebf17 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,13 +3,13 @@
 
 pkgname="ayatana-settings"
 pkgver="23.10.13"
-pkgrel=1
+pkgrel=2
 pkgdesc="Configuration tool for tweaking all Ayatana system indicators"
 arch=("any")
 url="https://github.com/AyatanaIndicators/ayatana-settings"
 license=("GPL3")
-depends=("gtk3" "gobject-introspection" "python-psutil")
-makedepends=("python-setuptools" "python-polib")
+depends=("gtk3" "gobject-introspection" "hicolor-icon-theme" "python" "python-psutil")
+makedepends=("python-build" "python-installer" "python-wheel" "python-setuptools" "python-polib")
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/AyatanaIndicators/${pkgname}/archive/${pkgver}.tar.gz")
 md5sums=("2a2433c4120e84af3e9b7fecf39062a6")
 options=("!emptydirs")
@@ -17,12 +17,12 @@ options=("!emptydirs")
 build()
 {
     cd ${srcdir}/${pkgname}-${pkgver}
-    python setup.py build
+    python -m build -w -n
 }
 
 package()
 {
     cd ${srcdir}/${pkgname}-${pkgver}
-    python setup.py install --root="${pkgdir}" --optimize=1
+    python -m installer -d "${pkgdir}" dist/*.whl
 }
 
