diff --git a/PKGBUILD b/PKGBUILD
index 7d14821..5018f81 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,13 +2,13 @@
 
 pkgname="ayatana-indicator-sound"
 pkgver="23.10.1"
-pkgrel="1"
+pkgrel="2"
 pkgdesc="Ayatana system sound indicator"
 arch=("i686" "x86_64" "pentium4")
 url="https://github.com/AyatanaIndicators/ayatana-indicator-sound"
 license=("GPL3")
-makedepends=("cmake-extras" "intltool" "vala" "systemd" "glib2" "libxml2" "gobject-introspection")
-depends=("accountsservice" "libpulse" "libgee" "libnotify" "libayatana-common>=0.9.9")
+makedepends=("cmake-extras" "intltool" "vala" "systemd" "libxml2" "gobject-introspection")
+depends=("accountsservice" "libpulse" "libgee" "libnotify" "libayatana-common>=0.9.9" "dconf" "glib2")
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/AyatanaIndicators/${pkgname}/archive/${pkgver}.tar.gz")
 md5sums=("34a35cf7f118f17e1a91336b871a7d5e")
 
