diff --git a/PKGBUILD b/PKGBUILD
index f8cdd38..c44db0d 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,8 +7,8 @@ pkgdesc="Ayatana system bluetooth indicator"
 arch=("i686" "x86_64" "pentium4")
 url="https://github.com/AyatanaIndicators/ayatana-indicator-bluetooth"
 license=("GPL3")
-makedepends=("cmake-extras" "intltool" "glib2" "systemd" "vala" )
-depends=("bluez" "blueman" "libayatana-common>=0.9.5")
+makedepends=("cmake-extras" "intltool" "systemd" "vala" )
+depends=("bluez" "blueman" "libayatana-common>=0.9.5" "dconf" "glib2")
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/AyatanaIndicators/${pkgname}/archive/${pkgver}.tar.gz")
 md5sums=("d168da67f12e49e8cca9f1f7404b016e")
 options=("!emptydirs")
