diff --git a/PKGBUILD b/PKGBUILD
index abca8f7..a3297bd 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,8 +7,8 @@ pkgdesc="Ayatana Indicator providing clock and calendar"
 arch=("i686" "x86_64" "pentium4")
 url="https://github.com/AyatanaIndicators/ayatana-indicator-datetime"
 license=("GPL3")
-makedepends=("cmake-extras" "ayatana-indicator-messages>=0.9.0" "intltool" "glib2" "libical" "properties-cpp")
-depends=("libayatana-common>=0.9.5" "evolution-data-server" "libaccounts-glib")
+makedepends=("cmake-extras" "ayatana-indicator-messages>=0.9.0" "intltool" "libical" "properties-cpp" "libnotify")
+depends=("libayatana-common>=0.9.5" "evolution-data-server" "libaccounts-glib" "dconf" "glib2")
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/AyatanaIndicators/${pkgname}/archive/${pkgver}.tar.gz")
 md5sums=("99664ac08ec27e2ae4a676b4324beffc")
 
