diff --git a/PKGBUILD b/PKGBUILD
index 024d5cd..940508a 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,13 +2,13 @@
 
 pkgname="ayatana-indicator-keyboard"
 pkgver="23.10.1"
-pkgrel="1"
+pkgrel="2"
 pkgdesc="Ayatana Indicator Keyboard Applet"
 arch=("i686" "x86_64" "pentium4")
 url="https://github.com/AyatanaIndicators/ayatana-indicator-keyboard"
 license=("GPL3")
-makedepends=("intltool" "cmake-extras" "systemd" "glib2" "libx11")
-depends=("libayatana-common>=0.9.9" "libxklavier" "hicolor-icon-theme" "accountsservice" "libxkbcommon")
+makedepends=("intltool" "cmake-extras" "systemd" "libx11")
+depends=("libayatana-common>=0.9.9" "libxklavier" "hicolor-icon-theme" "accountsservice" "libxkbcommon" "dconf" "glib2")
 optdepends=("matekbd-keyboard-display: Needed for displaying keyboard layouts on MATE" "libgnomekbd: Needed for displaying keyboard layouts on non-MATE")
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/AyatanaIndicators/${pkgname}/archive/${pkgver}.tar.gz")
 md5sums=("d739f4f6461590e34358f89faa7b7dd6")
