diff --git a/PKGBUILD b/PKGBUILD
index 49aa89b..fad2306 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,8 +7,8 @@ pkgdesc="Ayatana system sound indicator"
 arch=("i686" "x86_64" "pentium4")
 url="https://github.com/AyatanaIndicators/ayatana-indicator-sound"
 license=("GPL3")
-makedepends=("cmake-extras" "intltool" "vala" "systemd" "glib2" "libxml2" "gobject-introspection")
-depends=("accountsservice" "libpulse" "libgee" "libnotify" "libayatana-common>=0.9.5")
+makedepends=("cmake-extras" "intltool" "vala" "systemd" "libxml2" "gobject-introspection")
+depends=("accountsservice" "libpulse" "libgee" "libnotify" "libayatana-common>=0.9.5" "dconf" "glib2")
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/AyatanaIndicators/${pkgname}/archive/${pkgver}.tar.gz")
 md5sums=("2556587d1aa4e7d605e5fd27a86eb3ff")
 
