diff --git a/PKGBUILD b/PKGBUILD
index 770b622..a48ed29 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -12,21 +12,23 @@
 # `curl https://brave-browser-downloads.s3.brave.com/latest/release.version`
 
 pkgname=brave-bin
-pkgver=1.57.47
+pkgver=1.57.53
 pkgrel=1
 epoch=1
 pkgdesc='Web browser that blocks ads and trackers by default (binary release)'
 arch=(x86_64 aarch64)
 url=https://brave.com
 license=(MPL2 BSD custom:chromium)
-depends=(alsa-lib
-         gtk3
-         libxss
-         nss
-         ttf-font)
-optdepends=('cups: Printer support'
-            'libgnome-keyring: Enable GNOME keyring support'
-            'libnotify: Native notification support')
+depends=('alsa-lib' 'dbus' 'glib2' 'gtk3' 'hicolor-icon-theme' 'libxss' 'nss' 'ttf-font')
+optdepends=('cups: printer support'
+            'kdialog: support for native dialogs in Plasma'
+            'kwallet: support for storing passwords in KWallet on Plasma'
+            'libnotify: native notification support'
+            'org.freedesktop.secrets: password storage backend on GNOME / Xfce'
+            'pepper-flash: Adobe Flash Player support'
+            'pipewire: WebRTC desktop sharing under Wayland'
+            'qt5-base: enable Qt5 with --enable-features=AllowQt --qt-version=5'
+            'qt6-base: enable Qt6 with --enable-features=AllowQt --qt-version=6')
 provides=("${pkgname%-bin}=$pkgver" 'brave-browser')
 conflicts=("${pkgname%-bin}")
 options=(!strip)
@@ -37,8 +39,8 @@ source_aarch64=(${pkgname}-${pkgver}-aarch64.zip::https://github.com/brave/brave
 noextract=(${pkgname}-${pkgver}-x86_64.zip ${pkgname}-${pkgver}-aarch64.zip)
 sha256sums=('e7cd5f3d9b394d9add6b02f94cc78d6db589ae5d1ce06011bf69fa1f9b57365c'
             'c07276b69c7304981525ecb022f92daf7ae125a4fb05ac3442157b50826e257a')
-sha256sums_x86_64=('94306830b880d0783b41b2c9211a28faa914b1d2bb3dee57144588b3ec5e2ace')
-sha256sums_aarch64=('e760defd74a6adb929d7835c562484214703f90373b5dd3816029b22cf3d4742')
+sha256sums_x86_64=('e6a87c3ffb853e4ec2d2dcf8b4593d84759f48705684824e234b522a397f6768')
+sha256sums_aarch64=('1bb5989a80d0f2590c4f06f57a72c8b952643e71bbebe792bd2705177f93cd5f')
 
 prepare() {
 	mkdir -p brave
