diff --git a/PKGBUILD b/PKGBUILD
index b3ba3f5..de3bd06 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -12,21 +12,22 @@
 # `curl https://brave-browser-downloads.s3.brave.com/latest/release.version`
 
 pkgname=brave-bin
-pkgver=1.56.14
+pkgver=1.56.18
 pkgrel=1
 epoch=1
 pkgdesc='Web browser that blocks ads and trackers by default (binary release)'
 arch=(x86_64 aarch64)
 url=https://brave.com
 license=(MPL2 BSD custom:chromium)
-depends=(alsa-lib
-         gtk3
-         libxss
-         nss
-         ttf-font)
-optdepends=('cups: Printer support'
-            'libgnome-keyring: Enable GNOME keyring support'
-            'libnotify: Native notification support')
+depends=('alsa-lib' 'dbus' 'glib2' 'gtk3' 'hicolor-icon-theme' 'libxss' 'nss' 'ttf-font')
+optdepends=('cups: printer support'
+            'kdialog: support for native dialogs in Plasma'
+            'kwallet: support for storing passwords in KWallet on Plasma'
+            'libnotify: native notification support'
+            'org.freedesktop.secrets: password storage backend on GNOME / Xfce'
+            'pipewire: WebRTC desktop sharing under Wayland'
+            'qt5-base: enable Qt5 with --enable-features=AllowQt --qt-version=5'
+            'qt6-base: enable Qt6 with --enable-features=AllowQt --qt-version=6')
 provides=("${pkgname%-bin}=$pkgver" 'brave-browser')
 conflicts=("${pkgname%-bin}")
 options=(!strip)
@@ -37,8 +38,8 @@ source_aarch64=(${pkgname}-${pkgver}-aarch64.zip::https://github.com/brave/brave
 noextract=(${pkgname}-${pkgver}-x86_64.zip ${pkgname}-${pkgver}-aarch64.zip)
 sha256sums=('e7cd5f3d9b394d9add6b02f94cc78d6db589ae5d1ce06011bf69fa1f9b57365c'
             'c07276b69c7304981525ecb022f92daf7ae125a4fb05ac3442157b50826e257a')
-sha256sums_x86_64=('03263a26a105d937749e4362d01eb8262ed3c9ec00a63b2ba2b75faa71eebce3')
-sha256sums_aarch64=('2f76751da78ae171fe630d7c1af602903e66da9e237ed510c9b818c4763d9951')
+sha256sums_x86_64=('e295248ab519b45c592997ea99a1aefb5693305e712317aff6a1452d6a910e46')
+sha256sums_aarch64=('9a9c6ffe55ba8b001877ddea09f261e510aadb93e56b2f7f497485f9230b08e3')
 
 prepare() {
 	mkdir -p brave
