diff --git a/PKGBUILD b/PKGBUILD
index 3fd5444..9c88026 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,17 +1,17 @@
 # Maintainer: Emanuel Couto <unit73e@gmail.com>
 pkgname=cnijfilter2
-pkgver=6.50
-pkgrel=1
+pkgver=6.60
+pkgrel=2
 pkgdesc="Canon IJ Printer Driver for Linux"
 arch=('i686' 'x86_64')
 url="https://www.canon.com.au/home-printers"
 license=('GPL' 'custom:canon')
-depends=('cups' 'libxml2' 'ghostscript')
+depends=('cups' 'libxml2' 'ghostscript' 'libusb-1.0.so')
 makedepends=('automake' 'autoconf')
 provides=('tocanonij' 'tocnpwg' 'cnijlgmon3')
 conflicts=('cnijfilter' 'cnijfilter-mg3600')
-source=("https://gdlp01.c-wss.com/gds/9/0100011639/01/$pkgname-source-$pkgver-1.tar.gz")
-sha256sums=('1f240ee60adbd03f8706bd80f6f2f964b803facbb2e03f0ffbd68cc4ed77724e')
+source=("https://gdlp01.c-wss.com/gds/3/0100011753/01/$pkgname-source-$pkgver-1.tar.gz")
+b2sums=('0207784251e70c5850da74eb76a4de16d4dc5760e91ce17fc86ee9c61cfac27c041ea3f434ecad846ee3cc93649ac627d41eaaa55d3baf2e148ad36d8e142b87')
 
 prepare() {
 	cd "$pkgname-source-$pkgver-1"
@@ -24,14 +24,14 @@ build() {
 	pushd cmdtocanonij2
 	./autogen.sh --prefix=/usr \
 		     --datadir=/usr/share \
-		     LDFLAGS="-L../../com/libs_bin_$CARCH"
+		     LDFLAGS="${LDFLAGS} -L../../com/libs_bin_$CARCH"
 	make
 	popd
 
 	pushd cmdtocanonij3
 	./autogen.sh --prefix=/usr \
 		--datadir=/usr/share \
-		LDFLAGS="-L../../com/libs_bin_$CARCH"
+		LDFLAGS="${LDFLAGS} -L../../com/libs_bin_$CARCH"
 	make
 	popd
 
@@ -46,7 +46,7 @@ build() {
 		     --enable-libpath=/usr/lib/bjlib2 \
 		     --enable-progpath=/usr/bin \
 		     --datadir=/usr/share \
-		     LDFLAGS="-L../../com/libs_bin_$CARCH"
+		     LDFLAGS="${LDFLAGS} -L../../com/libs_bin_$CARCH"
 	make #CFLAGS="${CFLAGS} -fcommon"
 	popd
 
