diff --git a/PKGBUILD b/PKGBUILD
index c6cf0ad..97ba5ca 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,6 +1,6 @@
 # Maintainer: Emanuel Couto <unit73e@gmail.com>
 pkgname=cnijfilter2
-pkgver=6.30
+pkgver=6.40
 pkgrel=1
 pkgdesc="Canon IJ Printer Driver for Linux"
 arch=('i686' 'x86_64')
@@ -10,8 +10,8 @@ depends=('cups' 'libxml2' 'ghostscript')
 makedepends=('automake' 'autoconf')
 provides=('tocanonij' 'tocnpwg' 'cnijlgmon3')
 conflicts=('cnijfilter' 'cnijfilter-mg3600')
-source=("https://gdlp01.c-wss.com/gds/9/0100011319/01/$pkgname-source-$pkgver-1.tar.gz")
-md5sums=('f29bd241467f1e4bbe8b2c9c19923c8e')
+source=("https://gdlp01.c-wss.com/gds/1/0100011381/01/$pkgname-source-$pkgver-1.tar.gz")
+b2sums=('133751759e9c4b96a4e7b588f96b38dd3b021a328027faefe4c4fb75bf5cbddb2b3fd1020961cdbe654d665768a11ae38018b5e8915f0ca154d0983f14ae62f3')
 
 prepare() {
 	cd "$pkgname-source-$pkgver-1"
