diff --git a/PKGBUILD b/PKGBUILD
index 2f1bc61..4d9599c 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,17 +1,18 @@
 # Maintainer: Emanuel Couto <unit73e@gmail.com>
 pkgname=cnijfilter2
 pkgver=6.60
-pkgrel=1
+pkgrel=2
 pkgdesc="Canon IJ Printer Driver for Linux"
 arch=('i686' 'x86_64')
 url="https://www.canon.com.au/home-printers"
 license=('GPL' 'custom:canon')
-depends=('cups' 'libxml2' 'ghostscript')
+depends=('cups' 'libxml2' 'ghostscript' 'libusb-1.0.so')
 makedepends=('automake' 'autoconf')
 provides=('tocanonij' 'tocnpwg' 'cnijlgmon3')
 conflicts=('cnijfilter' 'cnijfilter-mg3600')
 source=("https://gdlp01.c-wss.com/gds/3/0100011753/01/$pkgname-source-$pkgver-1.tar.gz")
 sha256sums=('997119d67897d2b6405555bc6baa4cf7727bf0ecc6467f86df408831b1dab6d0')
+b2sums=('0207784251e70c5850da74eb76a4de16d4dc5760e91ce17fc86ee9c61cfac27c041ea3f434ecad846ee3cc93649ac627d41eaaa55d3baf2e148ad36d8e142b87')
 
 prepare() {
 	cd "$pkgname-source-$pkgver-1"
@@ -24,14 +25,14 @@ build() {
 	pushd cmdtocanonij2
 	./autogen.sh --prefix=/usr \
 		     --datadir=/usr/share \
-		     LDFLAGS="-L../../com/libs_bin_$CARCH"
+		     LDFLAGS="${LDFLAGS} -L../../com/libs_bin_$CARCH"
 	make
 	popd
 
 	pushd cmdtocanonij3
 	./autogen.sh --prefix=/usr \
 		--datadir=/usr/share \
-		LDFLAGS="-L../../com/libs_bin_$CARCH"
+		LDFLAGS="${LDFLAGS} -L../../com/libs_bin_$CARCH"
 	make
 	popd
 
@@ -46,7 +47,7 @@ build() {
 		     --enable-libpath=/usr/lib/bjlib2 \
 		     --enable-progpath=/usr/bin \
 		     --datadir=/usr/share \
-		     LDFLAGS="-L../../com/libs_bin_$CARCH"
+		     LDFLAGS="${LDFLAGS} -L../../com/libs_bin_$CARCH"
 	make #CFLAGS="${CFLAGS} -fcommon"
 	popd
 
