diff --git a/PKGBUILD b/PKGBUILD
index 9794183..1bcd9c0 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -6,10 +6,11 @@
 
 pkgname=ffmpeg
 pkgver=6.1
-pkgrel=2
+pkgrel=2.1
 epoch=2
 pkgdesc='Complete solution to record, convert and stream audio and video'
 arch=(x86_64)
+groups=('extra-rebuild')
 url=https://ffmpeg.org/
 license=(GPL3)
 depends=(
@@ -113,9 +114,6 @@ provides=(
   libswresample.so
   libswscale.so
 )
-options=(
-  debug
-)
 _tag=3cdfac27d3ea06f8719faed48b4ae2e75e94a463
 source=(
   git+https://git.ffmpeg.org/ffmpeg.git?signed#tag=${_tag}
