diff --git a/PKGBUILD b/PKGBUILD
index 1ea9a07..aac32dd 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -10,6 +10,7 @@ pkgrel=7
 epoch=2
 pkgdesc='Complete solution to record, convert and stream audio and video'
 arch=(x86_64)
+groups=('extra-rebuild')
 url=https://ffmpeg.org
 license=(GPL-3.0-only)
 depends=(
@@ -73,8 +74,8 @@ depends=(
   vid.stab
   vmaf
   vulkan-icd-loader
-  x264
-  x265
+  x264 libx264.so
+  x265 libx265.so
   xvidcore
   xz
   zimg
@@ -111,15 +112,12 @@ provides=(
   libswresample.so
   libswscale.so
 )
-options=(
-  debug
-)
 _tag=6f4048827982a8f48f71f551a6e1ed2362816eec
 source=(
   git+https://git.ffmpeg.org/ffmpeg.git?signed#tag=${_tag}
   add-av_stream_get_first_dts-for-chromium.patch
 )
-b2sums=('SKIP'
+b2sums=('9c19bd6b68d4224af2a93fdce199fd0ce9b00432c8818801d124ca993ac45a01d3a7618b66804c02c1f26787a5ca88d2d9f682f3e819cee8c2b6b3dc77e64052'
         '555274228e09a233d92beb365d413ff5c718a782008075552cafb2130a3783cf976b51dfe4513c15777fb6e8397a34122d475080f2c4483e8feea5c0d878e6de')
 validpgpkeys=(DD1EC9E8DE085C629B3E1846B18E8928B3948D64) # Michael Niedermayer <michael@niedermayer.cc>
 
@@ -254,3 +252,4 @@ package() {
 }
 
 # vim: ts=2 sw=2 et:
+pkgrel=${pkgrel}.1
