diff --git a/PKGBUILD b/PKGBUILD
index e0c1f80..b47027e 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -6,11 +6,12 @@
 
 pkgname=ffmpeg4.4
 pkgver=4.4.3
-pkgrel=2
+pkgrel=2.2
 pkgdesc='Complete solution to record, convert and stream audio and video'
 arch=(x86_64)
 url=https://ffmpeg.org/
 license=(GPL3)
+groups=('extra-rebuild')
 depends=(
   alsa-lib
   aom
@@ -107,6 +108,7 @@ pkgver() {
 prepare() {
   cd ffmpeg
   git cherry-pick -n 988f2e9eb063db7c1a678729f58aab6eba59a55b # fix nvenc on older gpus
+  git cherry-pick -n c3c8f97a9804b4234e97f13b0057ffc2c9af27c0
 }
 
 build() {
