diff --git a/PKGBUILD b/PKGBUILD
index 1aa0cf5..a5b38cd 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -23,13 +23,13 @@ pkgname=(
   gst-editing-services
   gstreamer-vaapi
   gst-python
-  gstreamer-docs
 )
 pkgver=1.24.0
 pkgrel=1
 pkgdesc="Multimedia graph framework"
 url="https://gstreamer.freedesktop.org/"
 arch=(x86_64)
+groups=('extra-rebuild')
 license=(LGPL-2.1-or-later)
 makedepends=(
   # superproject
@@ -60,7 +60,7 @@ makedepends=(
   webrtc-audio-processing-1 wildmidi wpewebkit zbar zvbi zxing-cpp
 
   # gst-plugins-ugly
-  a52dec libcdio libdvdread libmpeg2 x264
+  a52dec libcdio libdvdread libmpeg2 x264 libx264.so
 
   # gst-libav
   ffmpeg
@@ -85,6 +85,17 @@ validpgpkeys=(
   D637032E45B8C6585B9456565D2EEE6F6F349D7C # Tim Müller <tim@gstreamer-foundation.org>
 )
 
+# Exclude gstreamer-docs from sources, as rebuilding doesn't change anything
+for _num in $(seq 0 $((${#source[@]} - 1))); do
+  [[ ${source[${_num}]} =~ "gstreamer-docs" ]] && continue
+  source_nodocs+=("${source[${_num}]}")
+  b2sums_nodocs+=("${b2sums[${_num}]}")
+done
+
+source=("${source_nodocs[@]}")
+b2sums=("${b2sums_nodocs[@]}")
+unset source_nodocs b2sums_nodocs
+
 prepare() {
   cd gstreamer
 
@@ -93,6 +104,10 @@ prepare() {
 }
 
 build() {
+  # gstreamer/elements_leaks fails with =zero, but passes on =pattern
+  export CFLAGS="${CFLAGS/-ftrivial-auto-var-init=zero /-ftrivial-auto-var-init=pattern }"
+  export CXXFLAGS="${CXXFLAGS/-ftrivial-auto-var-init=zero /-ftrivial-auto-var-init=pattern }"
+
   local meson_options=(
     # Superproject options
     -D devtools=disabled
@@ -486,7 +501,7 @@ package_gst-plugins-bad() {
     libmpcdec libopenmpt librsvg libsndfile libsrtp libwebp libxml2 lilv
     mjpegtools neon nettle openal openexr openjpeg2 openssl opus pango qrencode
     rtmpdump sbc soundtouch spandsp srt svt-av1 svt-hevc
-    webrtc-audio-processing-1 wildmidi x265 zbar zvbi zxing-cpp
+    webrtc-audio-processing-1 wildmidi x265 libx265.so zbar zvbi zxing-cpp
   )
 
   cd root; local files=(
@@ -661,7 +676,7 @@ package_gst-plugins-ugly() {
   pkgdesc+=" - ugly plugins"
   depends=(
     "gst-plugins-base-libs=$pkgver"
-    a52dec libcdio libdvdread libmpeg2 x264
+    a52dec libcdio libdvdread libmpeg2 x264 libx264.so
   )
 
   cd root; local files=(
@@ -785,3 +800,4 @@ package_gstreamer-docs() {
 }
 
 # vim:set sw=2 sts=-1 et tw=80:
+pkgrel=${pkgrel}.1
