diff --git a/PKGBUILD b/PKGBUILD
index cd9dc6e..a030392 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -26,11 +26,12 @@ pkgname=(
   gstreamer-docs
 )
 pkgver=1.22.4
-pkgrel=1
+pkgrel=1.1
 pkgdesc="Multimedia graph framework"
 url="https://gstreamer.freedesktop.org/"
 arch=(x86_64)
 license=(LGPL)
+groups=('extra-rebuild')
 makedepends=(
   # superproject
   git meson
@@ -58,7 +59,7 @@ makedepends=(
   qrencode json-glib libva libxkbcommon-x11
 
   # gst-plugins-ugly
-  a52dec opencore-amr libcdio libdvdread libmpeg2 x264
+  a52dec opencore-amr libcdio libdvdread libmpeg2 libx264.so
 
   # gst-libav
   ffmpeg
@@ -91,6 +92,10 @@ prepare() {
 }
 
 build() {
+  # gstreamer/elements_leaks fails with =zero, but passes on =pattern
+  export CFLAGS="${CFLAGS/-ftrivial-auto-var-init=zero /-ftrivial-auto-var-init=pattern }"
+  export CXXFLAGS="${CXXFLAGS/-ftrivial-auto-var-init=zero /-ftrivial-auto-var-init=pattern }"
+
   local meson_options=(
     # Superproject options
     -D devtools=disabled
@@ -477,7 +482,7 @@ package_gst-plugins-bad() {
     libkate liblrdf lilv libmodplug mjpegtools libmpcdec neon openal openexr
     openjpeg2 opus libdvdnav libdvdread librsvg rtmpdump sbc libsndfile libltc
     soundtouch spandsp srt libsrtp zvbi libwebp webrtc-audio-processing wildmidi
-    x265 zbar gsm libopenmpt libldac libfreeaptx qrencode json-glib libavtp
+    libx265.so zbar gsm libopenmpt libldac libfreeaptx qrencode json-glib libavtp
     libmicrodns svt-hevc zxing-cpp
   )
 
@@ -634,7 +639,7 @@ package_gst-plugins-ugly() {
   pkgdesc+=" - ugly plugins"
   depends=(
     "gst-plugins-base-libs=$pkgver"
-    libdvdread libmpeg2 a52dec libcdio x264 opencore-amr
+    libdvdread libmpeg2 a52dec libcdio libx264.so opencore-amr
   )
 
   cd root; local files=(
