diff --git a/PKGBUILD b/PKGBUILD
index f660a91..421f436 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -26,11 +26,12 @@ pkgname=(
   gstreamer-docs
 )
 pkgver=1.22.5
-pkgrel=1
+pkgrel=1.1
 pkgdesc="Multimedia graph framework"
 url="https://gstreamer.freedesktop.org/"
 arch=(x86_64)
 license=(LGPL)
+groups=('extra-rebuild')
 makedepends=(
   # superproject
   git meson
@@ -58,7 +59,7 @@ makedepends=(
   webrtc-audio-processing wildmidi wpewebkit zbar zvbi zxing-cpp
 
   # gst-plugins-ugly
-  a52dec libcdio libdvdread libmpeg2 opencore-amr x264
+  a52dec libcdio libdvdread libmpeg2 opencore-amr libx264.so
 
   # gst-libav
   ffmpeg
@@ -96,6 +97,10 @@ prepare() {
 }
 
 build() {
+  # gstreamer/elements_leaks fails with =zero, but passes on =pattern
+  export CFLAGS="${CFLAGS/-ftrivial-auto-var-init=zero /-ftrivial-auto-var-init=pattern }"
+  export CXXFLAGS="${CXXFLAGS/-ftrivial-auto-var-init=zero /-ftrivial-auto-var-init=pattern }"
+
   local meson_options=(
     # Superproject options
     -D devtools=disabled
@@ -485,7 +490,7 @@ package_gst-plugins-bad() {
     libmpcdec libopenmpt librsvg libsndfile libsrtp libwebp libxml2 lilv
     mjpegtools neon nettle openal openexr openjpeg2 openssl opus pango qrencode
     rtmpdump sbc soundtouch spandsp srt svt-hevc webrtc-audio-processing
-    wildmidi x265 zbar zvbi zxing-cpp
+    wildmidi libx265.so zbar zvbi zxing-cpp
   )
 
   cd root; local files=(
@@ -656,7 +661,7 @@ package_gst-plugins-ugly() {
   pkgdesc+=" - ugly plugins"
   depends=(
     "gst-plugins-base-libs=$pkgver"
-    a52dec libcdio libdvdread libmpeg2 opencore-amr x264
+    a52dec libcdio libdvdread libmpeg2 opencore-amr libx264.so
   )
 
   cd root; local files=(
