diff --git a/PKGBUILD b/PKGBUILD
index 5da997e..00c06cf 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,10 +3,10 @@ pkgname=harmonyos-sans-git
 _pkgname=harmonyos-sans
 pkgver=r75.0d79cad
 pkgrel=1
-pkgdesc="鸿蒙黑体, HarmonyOS fonts"
+pkgdesc="HarmonyOS fonts"
 arch=('any')
 url="https://gitee.com/openharmony/resources"
-license=('custom')
+license=('custom:HarmonyOS Sans Fonts License Agreement')
 groups=()
 makedepends=('git')
 provides=("$_pkgname")
