diff --git a/PKGBUILD b/PKGBUILD
index 5010468..a7fc4fe 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,9 +9,11 @@ pkgdesc="companion package to llvm-minimal-git,  Library requirements of the Ope
 arch=('any')
 url="https://libclc.llvm.org/"
 license=('custom:Apache 2.0 with LLVM Exception')
+groups=('mesa-minimal')
 provides=(libclc)
 conflicts=(libclc)
-makedepends=(llvm-minimal-git clang-minimal-git cmake  git python spirv-llvm-translator-minimal-git 'llvm-libs<17')
+# edit makedepends array so we build with our llvm-git
+makedepends=(llvm-git cmake git python spirv-llvm-translator-minimal-git)
 source=("git+https://github.com/llvm/llvm-project.git"
 )
 sha256sums=('SKIP')
@@ -49,8 +51,13 @@ build() {
     cmake \
       -B _build \
       -S "$srcdir"/llvm-project/libclc  \
+      -D CMAKE_AR=/usr/bin/llvm-ar \
       -D CMAKE_BUILD_TYPE=Release \
+      -D CMAKE_C_COMPILER=clang \
+      -D CMAKE_CXX_COMPILER=clang++ \
       -D CMAKE_INSTALL_PREFIX=/usr \
+      -D CMAKE_RANLIB=/usr/bin/llvm-ranlib \
+      -D LLVM_USE_LINKER=lld \
       -Wno-dev
   make -C _build
 
