diff --git a/PKGBUILD b/PKGBUILD
index 0dd8210..5158364 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,13 +7,13 @@
 _jdkname=liberica-jdk-11-full
 pkgname="${_jdkname}-bin"
 _java_ver=11
-pkgver=11.0.19.u7
-_pkgver=11.0.19+7
+_pkgver=${_java_ver}.0.19+7
+pkgver=${_pkgver/+/.u}
 pkgrel=1
 pkgdesc='BellSoft builds of OpenJDK are fully certified and 100% open source Java Development Kits (JDKs) for all Java development and production workloads. Full version includes OpenJFX.'
 arch=(aarch64 armv7h armv8h x86_64)
 url='https://bell-sw.com/'
-license=('custom')
+license=('custom:GPL-2.0 with Classpath Exception' 'custom:liberica')
 depends=('java-environment-common=3' 'java-runtime-common>=3' 'ca-certificates-utils')
 optdepends=(ffmpeg freetype2 gtk2 gtk3 libnet)
 provides=(
