diff --git a/PKGBUILD b/PKGBUILD
index a4ca496..ddb3821 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,14 +7,15 @@
 _jdkname=liberica-jdk-11-full
 pkgname="${_jdkname}-bin"
 _java_ver=11
-pkgver=11.0.21.u10
-_pkgver=11.0.21+10
+_pkgver=${_java_ver}.0.21+10
+pkgver=${_pkgver/+/.u}
 pkgrel=1
-pkgdesc='BellSoft builds of OpenJDK are fully certified and 100% open source Java Development Kits (JDKs) for all Java development and production workloads. Full version includes OpenJFX.'
+pkgdesc='Free and open source Java Development Kit produced by BellSoft (full version with OpenJFX)'
 arch=(aarch64 armv7h armv8h x86_64)
 url='https://bell-sw.com/'
-license=('custom')
+license=('custom:liberica')
 depends=('java-environment-common=3' 'java-runtime-common>=3' 'ca-certificates-utils')
+depends+=('glib2') # namcap
 optdepends=(ffmpeg freetype2 gtk2 gtk3 libnet)
 provides=(
   "java-environment=$_java_ver"
@@ -74,9 +75,9 @@ package() {
 
   # Legal
   install -dm 755 "${pkgdir}/usr/share/licenses"
-  cp -r legal "${pkgdir}/usr/share/licenses/${_jdkname}"
+  cp -r legal "${pkgdir}/usr/share/licenses/${pkgname}"
   rm -rf "${pkgdir}/${_jvmdir}/legal"
-  ln -s "/usr/share/licenses/${_jdkname}" "${pkgdir}/${_jvmdir}/legal"
+  ln -s "/usr/share/licenses/${pkgname}" "${pkgdir}/${_jvmdir}/legal"
 
   # Man pages
   for f in man/man1/* man/ja/man1/*; do
