diff --git a/PKGBUILD b/PKGBUILD
index 2115f46..62d026c 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,13 +8,13 @@
 _jdkname=liberica-jdk-17-full
 pkgname="${_jdkname}-bin"
 _java_ver=17
-pkgver=${_java_ver}.0.4.1.u1
-_pkgver=${_java_ver}.0.4.1+1
+pkgver=${_java_ver}.0.5.u8
+_pkgver=${_java_ver}.0.5+8
 pkgrel=1
 pkgdesc='BellSoft builds of OpenJDK are fully certified and 100% open source Java Development Kits (JDKs) for all Java development and production workloads. Full version includes OpenJFX.'
 arch=(aarch64 armv7h armv8h x86_64)
 url='https://bell-sw.com/'
-license=('custom')
+license=('custom:GPL-2.0 with Classpath Exception' 'custom:liberica')
 depends=('java-environment-common=3' 'java-runtime-common>=3' 'ca-certificates-utils')
 optdepends=(ffmpeg freetype2 gtk2 gtk3 libnet)
 provides=(
@@ -69,10 +69,10 @@ source_armv8h=(${source_armv7h[@]})
 source_x86_64=(https://download.bell-sw.com/java/$_pkgver/bellsoft-jdk$_pkgver-linux-amd64-full.tar.gz)
 
 # Upstream-provided
-sha1sums_aarch64=('3b2d4229b11ea33739f4faf8eb97c027a3680c2d')
-sha1sums_armv7h=('b120af8e4686bc20eec6df4c898d483fa5692333')
+sha1sums_aarch64=('3b54eb34c0c19d706967c0019036d91ba383ab04')
+sha1sums_armv7h=('c53bc55c9218270da1db1436d54388fccd9fa17b')
 sha1sums_armv8h=(${sha1sums_armv7h[@]})
-sha1sums_x86_64=('ed4a451d82033604b55e474b5ce90b589c469970')
+sha1sums_x86_64=('3d38998904b408abb4384b99559e8d7cb765b3db')
 
 _jvmdir="/usr/lib/jvm/${_jdkname}"
 
