diff --git a/PKGBUILD b/PKGBUILD
index 33a19df..e0d5d68 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,14 +9,15 @@ _pkgbase=liberica-jdk-full
 _java_ver=17
 _jdkname="liberica-jdk-${_java_ver}-full"
 pkgname="${_jdkname}-bin"
-pkgver=${_java_ver}.0.8.u7
 _pkgver=${_java_ver}.0.8+7
-pkgrel=2
-pkgdesc='BellSoft builds of OpenJDK are fully certified and 100% open source Java Development Kits (JDKs) for all Java development and production workloads. Full version includes OpenJFX.'
+pkgver=${_pkgver/+/.u}
+pkgrel=1
+pkgdesc='Free and open source Java Development Kit produced by BellSoft (full version with OpenJFX)'
 arch=(aarch64 armv7h armv8h x86_64)
 url='https://bell-sw.com/'
-license=('custom')
+license=('custom:liberica')
 depends=('java-environment-common=3' 'java-runtime-common>=3' 'ca-certificates-utils')
+depends+=('hicolor-icon-theme' 'glib2') # namcap
 optdepends=(ffmpeg freetype2 gtk2 gtk3 libnet)
 provides=(
   "java-environment=${pkgver/.*}"
@@ -99,9 +100,9 @@ package() {
 
   # Legal
   install -dm 755 "${pkgdir}/usr/share/licenses"
-  cp -r legal "${pkgdir}/usr/share/licenses/${_jdkname}"
+  cp -r legal "${pkgdir}/usr/share/licenses/${pkgname}"
   rm -rf "${pkgdir}/${_jvmdir}/legal"
-  ln -s "/usr/share/licenses/${_jdkname}" "${pkgdir}/${_jvmdir}/legal"
+  ln -s "/usr/share/licenses/${pkgname}" "${pkgdir}/${_jvmdir}/legal"
 
   # Man pages
   for f in man/man1/*; do
