diff --git a/PKGBUILD b/PKGBUILD
index 99099d5..fd80075 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,12 +2,16 @@
 
 pkgname=libglibutil
 pkgver=1.0.76
-pkgrel=1
+pkgrel=2
 pkgdesc="Library of glib utilities"
 arch=('x86_64' 'i686' 'armv7h' 'aarch64')
 url="https://github.com/sailfishos/libglibutil"
-license=('BSD')
-depends=('glib2')
+license=('BSD-3-Clause')
+depends=(
+  'glib2' 'libglib-2.0.so' 'libgobject-2.0.so'
+  'glibc'
+)
+provides=('libglibutil.so')
 source=("$pkgname-$pkgver.tar.gz::https://github.com/sailfishos/libglibutil/archive/refs/tags/${pkgver}.tar.gz")
 sha512sums=('f0b35f0366889b6a4c83d73e416a50022f2f0efc08f0fd34b6458c2e8c8b4cabeda6ebbcd3ce9682ab7b92640d956a534683aa2d6773771914ed5dbfb1ad99ee')
 
@@ -19,4 +23,5 @@ build() {
 package() {
   cd $pkgname-$pkgver
   make install-dev DESTDIR="${pkgdir}"
+  install -Dm644 LICENSE "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
 }
