diff --git a/PKGBUILD b/PKGBUILD
index 6055012..8cb0885 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Danct12 <danct12@disroot.org>
 
 pkgname=libgbinder
-pkgver=1.1.32
+pkgver=1.1.34
 pkgrel=1
 pkgdesc="GLib-style interface to binder"
 arch=('x86_64' 'i686' 'armv7h' 'aarch64')
@@ -9,7 +9,7 @@ url="https://github.com/mer-hybris/libgbinder.git"
 license=('BSD')
 depends=('libglibutil' 'glib2')
 makedepends=('git')
-_commit="84efc32ac09925b166843a7a60592c378689ccd3" # tags/1.1.32
+_commit="62b9b30865b65886a5808ab8087194fcca163c90" # tags/1.1.34
 source=(${pkgname}::git+https://github.com/mer-hybris/libgbinder.git#commit=${_commit})
 sha512sums=('SKIP')
 
