diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 0de0a0d..713b1d4 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -3,13 +3,13 @@
 
 pkgname=libheif
 pkgver=1.16.1
-pkgrel=2
+pkgrel=2.1
 pkgdesc='HEIF file format decoder and encoder'
 arch=(x86_64)
 url='https://github.com/strukturag/libheif'
 license=(GPL3)
 makedepends=(cmake libjpeg libpng gdk-pixbuf2 svt-av1 rav1e)
-depends=(libde265 x265 aom dav1d)
+depends=(libde265 libx265.so aom dav1d)
 optdepends=('libjpeg: for heif-convert and heif-enc' 'libpng: for heif-convert and heif-enc'
             'svt-av1: svt-av1 encoder' 'rav1e: rav1e encoder')
 source=(https://github.com/strukturag/libheif/releases/download/v$pkgver/libheif-$pkgver.tar.gz)
