diff --git a/PKGBUILD b/PKGBUILD
index b188a33..43ec683 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,13 +3,14 @@
 
 pkgname=libheif
 pkgver=1.16.2
-pkgrel=1
+pkgrel=1.1
 pkgdesc='HEIF file format decoder and encoder'
 arch=(x86_64)
 url='https://github.com/strukturag/libheif'
 license=(GPL3)
+groups=('extra-rebuild')
 makedepends=(cmake libjpeg libpng gdk-pixbuf2 svt-av1 rav1e)
-depends=(libde265 x265 aom dav1d)
+depends=(libde265 libx265.so aom dav1d)
 optdepends=('libjpeg: for heif-convert and heif-enc' 'libpng: for heif-convert and heif-enc'
             'svt-av1: svt-av1 encoder' 'rav1e: rav1e encoder')
 source=(https://github.com/strukturag/libheif/releases/download/v$pkgver/libheif-$pkgver.tar.gz)
