diff --git a/PKGBUILD b/PKGBUILD
index 16a2de2..d45763a 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,7 +7,8 @@ pkgrel=5
 pkgdesc='An HEIF and AVIF file format decoder and encoder'
 arch=(x86_64)
 url='https://github.com/strukturag/libheif'
-license=(GPL3)
+groups=('extra-rebuild')
+license=('LGPL-3.0-or-later')
 makedepends=(cmake
              dav1d
              ffmpeg
@@ -21,7 +22,7 @@ depends=(aom
          glibc
          libde265
          libwebp
-         x265)
+         x265 libx265.so)
 optdepends=('libjpeg: for heif-convert and heif-enc'
             'libpng: for heif-convert and heif-enc'
             'dav1d: dav1d encoder'
@@ -54,3 +55,4 @@ build() {
 package() {
   DESTDIR="$pkgdir" cmake --install build
 }
+pkgrel=${pkgrel}.1
