diff --git a/PKGBUILD b/PKGBUILD
index 0046d3e..ae42cd6 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,10 +3,11 @@
 
 pkgname=libheif
 pkgver=1.17.6
-pkgrel=1
+pkgrel=1.1
 pkgdesc='An HEIF and AVIF file format decoder and encoder'
 arch=(x86_64)
 url='https://github.com/strukturag/libheif'
+groups=('extra-rebuild')
 license=(GPL3)
 makedepends=(cmake
              dav1d
@@ -21,7 +22,7 @@ depends=(aom
          glibc
          libde265
          libwebp
-         x265)
+         x265 libx265.so)
 optdepends=('libjpeg: for heif-convert and heif-enc'
             'libpng: for heif-convert and heif-enc'
             'dav1d: dav1d encoder'
