diff --git a/PKGBUILD b/PKGBUILD
index 532ecc8..3522cc6 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,12 +1,12 @@
 # Maintainer: Felix Kauselmann <licorn@gmail.com>
 
 pkgname=libpdfium-nojs
-pkgver=5060.r1.558516c323
+pkgver=5060.r2.57e40e8fe1
 pkgrel=1
 pkgdesc="Open-source PDF rendering engine."
 arch=('x86_64')
 url="https://pdfium.googlesource.com/pdfium/"
-license=('BSD')
+license=('custom:BSD')
 depends=('freetype2' 'lcms2' 'libjpeg' 'openjpeg2' 'icu')
 conflicts=('libpdfium-bin')
 provides=('libpdfium')
