diff --git a/PKGBUILD b/PKGBUILD
index 5fdd819..5c185d9 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,20 +1,32 @@
 # Maintainer: Felix Kauselmann <licorn@gmail.com>
 
 pkgname=libpdfium-nojs
-pkgver=6261.r0.4c4f9ab25d
+pkgver=6312.r0.g812acaddb5
 pkgrel=1
 pkgdesc="Open-source PDF rendering engine."
 arch=('x86_64')
 url="https://pdfium.googlesource.com/pdfium/"
-license=('BSD')
-depends=('freetype2' 'lcms2' 'libjpeg' 'openjpeg2' 'icu')
-conflicts=('libpdfium-bin')
+license=(
+  'Apache-2.0'
+  'BSD-3-Clause'
+)
+depends=(
+  'freetype2' 'libfreetype.so'
+  'gcc-libs'
+  'glibc'
+  'icu' 'libicuuc.so'
+  'lcms2' 'liblcms2.so'
+  'libjpeg' 'libjpeg.so'
+  'openjpeg2'
+  'zlib' 'libz.so'
+)
+conflicts=('libpdfium')
 provides=('libpdfium')
 makedepends=('git' 'python' 'gn' 'ninja')
 
-source=("git+https://pdfium.googlesource.com/pdfium"
-    "git+https://chromium.googlesource.com/chromium/src/build.git"
-    "git+https://chromium.googlesource.com/chromium/src/third_party/abseil-cpp"
+source=("pdfium::git+https://pdfium.googlesource.com/pdfium"
+    "chromium-build::git+https://chromium.googlesource.com/chromium/src/build.git"
+    "chromium-abseil-cpp::git+https://chromium.googlesource.com/chromium/src/third_party/abseil-cpp"
     "libpdfium.pc"
     )
 
@@ -29,7 +41,7 @@ pkgver() {
   
   # Version = branch name/number + number of commits since branch creation
   # + short head
-  printf  "%s.r%s.%s" $(git rev-parse --abbrev-ref HEAD | cut -d '/' -f2)\
+  printf  "%s.r%s.g%s" $(git rev-parse --abbrev-ref HEAD | cut -d '/' -f2)\
    $(git rev-list --count main..) $(git rev-parse --short HEAD)
 
 }
@@ -38,8 +50,8 @@ prepare() {
 
   cd "$srcdir/pdfium"
 
-  ln -sf $srcdir/build build
-  ln -sf $srcdir/abseil-cpp third_party/abseil-cpp
+  ln -sf $srcdir/chromium-build build
+  ln -sf $srcdir/chromium-abseil-cpp third_party/abseil-cpp
 
   # Pdfium is developed alongside Chromium and does not provide releases
   # Upstream recommends using Chromium's dev channels instead
@@ -67,7 +79,7 @@ prepare() {
 
   # Use system provided icu library (unbundling)
   mkdir -p "$srcdir/pdfium/third_party/icu"
-  ln -sf "$srcdir/build/linux/unbundle/icu.gn" "$srcdir/pdfium/third_party/icu/BUILD.gn"
+  ln -sf "$srcdir/chromium-build/linux/unbundle/icu.gn" "$srcdir/pdfium/third_party/icu/BUILD.gn"
 
   # Download and decode shim header script needed to unbundle icu (gittiles is weird)
   mkdir -p "$srcdir/pdfium/tools/generate_shim_headers/"
@@ -77,7 +89,7 @@ prepare() {
   echo "Done."
   
   # Create fake gclient_args.gni file to satisfy include list for build/config/compiler/compiler.gni
-  touch "$srcdir/build/config/gclient_args.gni"
+  touch "$srcdir/chromium-build/config/gclient_args.gni"
   
   # Exclude test fonts from build
   cd "$srcdir/pdfium/testing/"
