diff --git a/PKGBUILD b/PKGBUILD
index 5f16bcf..87d7b90 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,14 +7,15 @@ pkgdesc="Open-source PDF rendering engine."
 arch=('x86_64')
 url="https://pdfium.googlesource.com/pdfium/"
 license=('BSD')
-depends=('freetype2' 'lcms2' 'libjpeg' 'openjpeg2' 'icu')
-conflicts=('libpdfium-bin')
+# https://aur.archlinux.org/packages/libpdfium-nojs#comment-809341 + namcap
+depends=('freetype2' 'lcms2' 'libjpeg' 'openjpeg2' 'icu' 'libicuuc.so' 'zlib')
+conflicts=('libpdfium')
 provides=('libpdfium')
 makedepends=('git' 'python' 'gn' 'ninja')
 
-source=("git+https://pdfium.googlesource.com/pdfium"
-    "git+https://chromium.googlesource.com/chromium/src/build.git"
-    "git+https://chromium.googlesource.com/chromium/src/third_party/abseil-cpp"
+source=("pdfium::git+https://pdfium.googlesource.com/pdfium"
+    "chromium-build::git+https://chromium.googlesource.com/chromium/src/build.git"
+    "chromium-abseil-cpp::git+https://chromium.googlesource.com/chromium/src/third_party/abseil-cpp"
     "libpdfium.pc"
     )
 
@@ -38,8 +39,8 @@ prepare() {
 
   cd "$srcdir/pdfium"
 
-  ln -sf $srcdir/build build
-  ln -sf $srcdir/abseil-cpp third_party/abseil-cpp
+  ln -sf $srcdir/chromium-build build
+  ln -sf $srcdir/chromium-abseil-cpp third_party/abseil-cpp
 
   # Pdfium is developed alongside Chromium and does not provide releases
   # Upstream recommends using Chromium's dev channels instead
@@ -65,7 +66,7 @@ prepare() {
 
   # Use system provided icu library (unbundling)
   mkdir -p "$srcdir/pdfium/third_party/icu"
-  ln -sf "$srcdir/build/linux/unbundle/icu.gn" "$srcdir/pdfium/third_party/icu/BUILD.gn"
+  ln -sf "$srcdir/chromium-build/linux/unbundle/icu.gn" "$srcdir/pdfium/third_party/icu/BUILD.gn"
 
   # Download and decode shim header script needed to unbundle icu (gittiles is weird)
   mkdir -p "$srcdir/pdfium/tools/generate_shim_headers/"
@@ -75,7 +76,7 @@ prepare() {
   echo "Done."
   
   # Create fake gclient_args.gni file to satisfy include list for build/config/compiler/compiler.gni
-  touch "$srcdir/build/config/gclient_args.gni"
+  touch "$srcdir/chromium-build/config/gclient_args.gni"
   
   # Exclude test fonts from build
   cd "$srcdir/pdfium/testing/"
