diff --git a/PKGBUILD b/PKGBUILD
index d91db90..8536e15 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,7 +8,8 @@ pkgdesc="a collection of common functions, classes and tools. Mainly used by w_s
 arch=('i686' 'x86_64' 'armv6h' 'armv7h')
 url="https://github.com/wirbel-at-vdr-portal/librepfunc"
 license=('GPL2')
-depends=('glibc' 'gcc-libs')
+depends=('gcc-libs')
+options=('!emptydirs')
 source=("$pkgname-$pkgver.tar.gz::https://github.com/wirbel-at-vdr-portal/librepfunc/archive/refs/tags/$pkgver.tar.gz")
 sha256sums=('ea0c68fd698aa4660fe870906ce2128e7925135d5abaaf19cd80f5054d177148')
 
