diff --git a/PKGBUILD b/PKGBUILD
index 3d6cb71..a2e66c3 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -6,11 +6,21 @@ pkgbase=linux-firmware-git
 pkgname=(linux-firmware-whence-git linux-firmware-git amd-ucode-git
          linux-firmware-{nfp,mellanox,marvell,qcom,liquidio,qlogic,bnx2x}-git
 )
-pkgver=20231215.c156e6be
+pkgver=20240203.b4b04a5c
 pkgrel=1
 pkgdesc="Firmware files for Linux"
 url="https://git.kernel.org/?p=linux/kernel/git/firmware/linux-firmware.git;a=summary"
-license=('GPL2' 'GPL3' 'custom')
+# Many firmware files are licensed under a variant of BSD-style license that only permits unmodified
+# binary redistribution for use in officially licensed or authorized products by the manufacturer.
+# Some others in particular are licensed under a version of Intel Simplified License or its
+# modifications, which is based on BSD-2-Clause license but has an additional clause that prohibits
+# any attempt to reverse engineer the firmware and thus making it non-free, besides granting limited
+# patent rights. The rest of the provided non-free firmware files are licensed under a custom EULA
+# that grants no or limited patent rights. Only a few known firmware files are actually licensed
+# under open source license or allegedly so, but may not be considered fully free either.
+#
+# The following SPDX identifers are listed in best effort attempt.
+license=('BSD-3-Clause AND (BSD-3-Clause OR GPL-2.0-only) AND BSD-3-Clause-Clear AND BSD-4-Clause AND BSD-Source-Code AND GPL-2.0-only AND (GPL-2.0-only OR Linux-OpenIB) AND (GPL-2.0-only OR MPL-1.1) AND GPL-2.0-or-later AND GPL-2.0-or-later WITH eCos-exception-2.0 AND GPL-3.0-only AND ISC AND MIT AND LicenseRef-Redistributable AND LicenseRef-Unknown')
 arch=('any')
 makedepends=('git' 'rdfind')
 options=(!strip)
@@ -61,7 +71,7 @@ package_linux-firmware-whence-git() {
   provides=("${pkgname%-git}=$pkgver")
   pkgdesc+=" - contains the WHENCE license file which documents the vendor license details"
 
-  install -Dt "${pkgdir}/usr/share/licenses/${pkgname%-git}" -m644 ${pkgbase}/WHENCE
+  install -Dt "${pkgdir}/usr/share/licenses/${pkgname}" -m644 ${pkgbase}/WHENCE
 }
 
 package_linux-firmware-git() {
@@ -73,7 +83,7 @@ package_linux-firmware-git() {
 
   ZSTD_CLEVEL=19 make DESTDIR="${pkgdir}" FIRMWAREDIR=/usr/lib/firmware install-zst
 
-  install -Dt "${pkgdir}/usr/share/licenses/${pkgname%-git}" -m644 LICEN*
+  install -Dt "${pkgdir}/usr/share/licenses/${pkgname}" -m644 LICEN*
 
   cd "${pkgdir}"
 
@@ -81,22 +91,24 @@ package_linux-firmware-git() {
   rm usr/lib/firmware/mrvl/prestera/mvsw_prestera_fw_arm64-v4.1.img.zst
 
   # split
+  _pick amd-ucode usr/lib/firmware/amd-ucode
+
   _pick linux-firmware-nfp usr/lib/firmware/netronome
-  _pick linux-firmware-nfp usr/share/licenses/${pkgname%-git}/LICENCE.Netronome
+  _pick linux-firmware-nfp usr/share/licenses/${pkgname}/LICENCE.Netronome
 
   _pick linux-firmware-mellanox usr/lib/firmware/mellanox
 
   _pick linux-firmware-marvell usr/lib/firmware/{libertas,mwl8k,mwlwifi,mrvl}
-  _pick linux-firmware-marvell usr/share/licenses/${pkgname%-git}/LICENCE.{Marvell,NXP}
+  _pick linux-firmware-marvell usr/share/licenses/${pkgname}/LICENCE.{Marvell,NXP}
 
   _pick linux-firmware-qcom usr/lib/firmware/{qcom,a300_*}
-  _pick linux-firmware-qcom usr/share/licenses/${pkgname%-git}/LICENSE.qcom*
+  _pick linux-firmware-qcom usr/share/licenses/${pkgname}/LICENSE.qcom*
 
   _pick linux-firmware-liquidio usr/lib/firmware/liquidio
-  _pick linux-firmware-liquidio usr/share/licenses/${pkgname%-git}/LICENCE.cavium_liquidio
+  _pick linux-firmware-liquidio usr/share/licenses/${pkgname}/LICENCE.cavium_liquidio
 
   _pick linux-firmware-qlogic usr/lib/firmware/{qlogic,qed,ql2???_*,c{b,t,t2}fw-*}
-  _pick linux-firmware-qlogic usr/share/licenses/${pkgname%-git}/LICENCE.{qla1280,qla2xxx}
+  _pick linux-firmware-qlogic usr/share/licenses/${pkgname}/LICENCE.{qla1280,qla2xxx}
 
   _pick linux-firmware-bnx2x usr/lib/firmware/bnx2x*
 }
@@ -105,17 +117,20 @@ package_amd-ucode-git() {
   conflicts=("${pkgname%-git}")
   provides=("${pkgname%-git}=$pkgver")
   pkgdesc="Microcode update image for AMD CPUs"
-  license=(custom)
+  license=('LicenseRef-Redistributable')
+
+  mv -v amd-ucode/* "${pkgdir}"
 
   install -Dt "${pkgdir}/boot" -m644 amd-ucode.img
 
-  install -Dt "${pkgdir}/usr/share/licenses/${pkgname%-git}" -m644 ${pkgbase}/LICENSE.amd-ucode
+  install -Dt "${pkgdir}/usr/share/licenses/${pkgname}" -m644 ${pkgbase}/LICENSE.amd-ucode
 }
 
 package_linux-firmware-nfp-git() {
   conflicts=("${pkgname%-git}" 'linux-firmware<=20211216.f682ecb')
   provides=("${pkgname%-git}=$pkgver")
   pkgdesc+=" - nfp / Firmware for Netronome Flow Processors"
+  license=('LicenseRef-Redistributable')
   depends=('linux-firmware-whence')
 
   mv -v linux-firmware-nfp/* "${pkgdir}"
@@ -125,6 +140,7 @@ package_linux-firmware-mellanox-git() {
   conflicts=("${pkgname%-git}" 'linux-firmware<=20211216.f682ecb')
   provides=("${pkgname%-git}=$pkgver")
   pkgdesc+=" - mellanox / Firmware for Mellanox Spectrum switches"
+  license=('BSD-3-Clause OR GPL-2.0-only')
   depends=('linux-firmware-whence')
 
   mv -v linux-firmware-mellanox/* "${pkgdir}"
@@ -134,6 +150,7 @@ package_linux-firmware-marvell-git() {
   conflicts=("${pkgname%-git}" 'linux-firmware<=20211216.f682ecb')
   provides=("${pkgname%-git}=$pkgver")
   pkgdesc+=" - marvell / Firmware for Marvell devices"
+  license=('LicenseRef-Redistributable')
   depends=('linux-firmware-whence')
 
   mv -v linux-firmware-marvell/* "${pkgdir}"
@@ -143,6 +160,7 @@ package_linux-firmware-qcom-git() {
   conflicts=("${pkgname%-git}" 'linux-firmware<=20211216.f682ecb')
   provides=("${pkgname%-git}=$pkgver")
   pkgdesc+=" - qcom / Firmware for Qualcomm SoCs"
+  license=('BSD-3-Clause AND LicenseRef-Redistributable')
   depends=('linux-firmware-whence')
 
   mv -v linux-firmware-qcom/* "${pkgdir}"
@@ -152,6 +170,7 @@ package_linux-firmware-liquidio-git() {
   conflicts=("${pkgname%-git}" 'linux-firmware<=20211216.f682ecb')
   provides=("${pkgname%-git}=$pkgver")
   pkgdesc+=" - liquidio / Firmware for Cavium LiquidIO server adapters"
+  license=('LicenseRef-Redistributable')
   depends=('linux-firmware-whence')
 
   mv -v linux-firmware-liquidio/* "${pkgdir}"
@@ -161,6 +180,7 @@ package_linux-firmware-qlogic-git() {
   conflicts=("${pkgname%-git}" 'linux-firmware<=20211216.f682ecb')
   provides=("${pkgname%-git}=$pkgver")
   pkgdesc+=" - qlogic / Firmware for QLogic devices"
+  license=('BSD-3-Clause AND LicenseRef-Redistributable')
   depends=('linux-firmware-whence')
 
   mv -v linux-firmware-qlogic/* "${pkgdir}"
@@ -170,6 +190,7 @@ package_linux-firmware-bnx2x-git() {
   conflicts=("${pkgname%-git}" 'linux-firmware<=20211216.f682ecb')
   provides=("${pkgname%-git}=$pkgver")
   pkgdesc+=" - bnx2x / Firmware for Broadcom NetXtreme II 10Gb ethernet adapters"
+  license=('LicenseRef-Redistributable') # is this correct?
   depends=('linux-firmware-whence')
 
   mv -v linux-firmware-bnx2x/* "${pkgdir}"
