diff --git a/PKGBUILD b/PKGBUILD
index 9a09288..130a2d2 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -15,16 +15,22 @@
 # Contributor: Tomas Wilhelmsson <tomas.wilhelmsson@gmail.com>
 
 
+# pin commit that passes Debian/Ubuntu expensive CI checks
+_commit=ffe41819e58365dfbe85a22556c0d9d284e746b9
+
 pkgbase=llvm-minimal-git
 pkgname=(llvm-minimal-git llvm-libs-minimal-git clang-minimal-git clang-libs-minimal-git clang-opencl-headers-minimal-git)
-pkgver=19.0.0_r490673.1901f442ca63
-pkgrel=2
+pkgver=19.0.0.r492391.gffe41819e583
+pkgrel=1
 arch=('x86_64')
 url="https://llvm.org/"
-license=('custom:Apache 2.0 with LLVM Exception')
+license=('Apache-2.0 WITH LLVM-exception')
+groups=('mesa-minimal')
 makedepends=(git cmake libffi libedit ncurses libxml2
              libxcrypt python python-setuptools)
-source=("git+https://github.com/llvm/llvm-project.git"
+# Build with LLVM instead of GCC, using llvm-git (expected of same version)
+makedepends+=('llvm-git')
+source=("llvm-project::git+https://github.com/llvm/llvm-project.git#commit=${_commit}"
 )
                 
 md5sums=('SKIP')
@@ -42,7 +48,7 @@ options=('!lto')
 # LITFLAGS="-j 18"
 # NOTE: It's your responbility to validate the value of LITFLAGS. If unsure, don't set it.
 
-_major_ver=19
+_major_ver=${pkgver%%.*}
 
 _get_distribution_components() {
 local target
@@ -87,7 +93,7 @@ done
 
 
 pkgver() {
-    cd llvm-project/llvm
+    cd llvm-project/cmake/Modules
 
     # This will almost match the output of `llvm-config --version` when the
     # LLVM_APPEND_VC_REV cmake flag is turned on. The only difference is
@@ -96,7 +102,7 @@ pkgver() {
             'BEGIN { ORS="." ; i=0 } \
              /set\(LLVM_VERSION_/ { print $2 ; i++ ; if (i==2) ORS="" } \
              END { print "\n" }' \
-             CMakeLists.txt)_r$(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+             LLVMVersion.cmake).r$(git rev-list --count HEAD).g$(git rev-parse --short HEAD)
     echo "${_pkgver}"
 }
 
@@ -104,9 +110,15 @@ build() {
     
     local cmake_args=(
         -G "Unix Makefiles"
+        -D CMAKE_AR=/usr/bin/llvm-ar
         -D CMAKE_BUILD_TYPE=Release
+        -D CMAKE_C_COMPILER=clang
+        -D CMAKE_CXX_COMPILER=clang++
         -D CMAKE_INSTALL_PREFIX=/usr
+        -D CMAKE_RANLIB=/usr/bin/llvm-ranlib
         -D LLVM_BINUTILS_INCDIR=/usr/include
+        -D LLVM_ENABLE_LTO=Thin
+        -D LLVM_USE_LINKER=lld
         -D FFI_INCLUDE_DIR=$(pkg-config --variable=includedir libffi)
         -D LLVM_VERSION_SUFFIX=""
         -D LLVM_APPEND_VC_REV=ON
@@ -154,7 +166,14 @@ check() {
 
 package_llvm-minimal-git() {
     pkgdesc="Collection of modular and reusable compiler and toolchain technologies"
-    depends=(llvm-libs-minimal-git="$pkgver-$pkgrel")
+    depends=(
+        "llvm-libs-minimal-git=${pkgver}-${pkgrel}"
+        'gcc-libs'
+        'glibc'
+        'ncurses' 'libncursesw.so'
+        'zlib' 'libz.so'
+        'zstd' 'libzstd.so'
+    )
     provides=('llvm')
     conflicts=('llvm')
     optdepends=('python: for using lit (LLVM Integrated Tester)'
@@ -203,7 +222,17 @@ package_llvm-minimal-git() {
 
 package_llvm-libs-minimal-git() {
     pkgdesc="LLVM runtime libraries, trunk version"
-    depends=(gcc-libs zlib libffi libedit ncurses libxml2 llvm-libs)
+    depends=(
+        "llvm-libs<${_major_ver}"
+        'gcc-libs'
+        'glibc'
+        'libedit' 'libedit.so'
+        'libffi' 'libffi.so'
+        'libxml2' 'libxml2.so'
+        'ncurses' 'libncursesw.so'
+        'zlib' 'libz.so'
+        'zstd' 'libzstd.so'
+    )
     # some applications expect llvmgold.so to be present and fail badly if it isn't.
     # adding repo llvm-libs as depend is the easiest solution
 
@@ -220,7 +249,15 @@ _python_optimize() {
 
 package_clang-minimal-git(){
   pkgdesc='C language family frontend for LLVM (git version)'
-  depends=(llvm-libs-minimal-git clang-libs-minimal-git gcc)
+  depends=(
+      "clang-libs-minimal-git=${pkgver}-${pkgrel}"
+      "llvm-libs-minimal-git=${pkgver}-${pkgrel}"
+      'gcc-libs'
+      'glibc'
+      'libxml2' 'libxml2.so'
+      'ncurses' 'libncursesw.so'
+      'perl'
+  )
   optdepends=('openmp: OpenMP support in clang with -fopenmp'
               'python: for scan-view, scan-build, git-clang-format, clang-rename and python bindings'
               'llvm-minimal-git: referenced by some clang headers')
@@ -249,7 +286,11 @@ package_clang-minimal-git(){
 
 package_clang-libs-minimal-git() {
     pkgdesc="clang runtime libraries, trunk version"
-    depends=(llvm-libs-minimal-git="$pkgver-$pkgrel")
+    depends=(
+        "llvm-libs-minimal-git=${pkgver}-${pkgrel}"
+        'gcc-libs'
+        'glibc'
+    )
     # the functionality offered by this package is part of the clang repo pacakge.
     # TODO: when/if this functionality is split off from repo clang, verify if changes are needed to this package
 
