diff --git a/PKGBUILD b/PKGBUILD
index 1e1308b..552e6de 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -17,14 +17,18 @@
 
 pkgbase=llvm-minimal-git
 pkgname=(llvm-minimal-git llvm-libs-minimal-git clang-minimal-git clang-libs-minimal-git)
-pkgver=17.0.0_r460868.6e19eea02bbe
+pkgver=17.0.0_r464764.01b512882d75
 pkgrel=1
 arch=('x86_64')
 url="https://llvm.org/"
 license=('custom:Apache 2.0 with LLVM Exception')
+groups=('mesa-minimal')
 makedepends=(git cmake libffi libedit ncurses libxml2
              libxcrypt python python-setuptools)
-source=("git+https://github.com/llvm/llvm-project.git"
+# Build with LLVM instead of GCC, using llvm-git (expected of same version)
+makedepends+=('llvm-git')
+# pin commit that passes Debian/Ubuntu expensive CI checks - 17 June 2023, 05:47 AM UTC
+source=("llvm-project::git+https://github.com/llvm/llvm-project.git#commit=01b512882d75c5fd64ca0e78f28acd3f0e2254d8"
                 'local://llvm-config.h'
 )
                 
@@ -64,9 +68,14 @@ build() {
     cmake \
         -B _build \
         -S "$srcdir"/llvm-project/llvm  \
+        -D CMAKE_AR=/usr/bin/llvm-ar \
         -D CMAKE_BUILD_TYPE=Release \
+        -D CMAKE_C_COMPILER=clang \
+        -D CMAKE_CXX_COMPILER=clang++ \
         -D CMAKE_INSTALL_PREFIX=/usr \
+        -D CMAKE_RANLIB=/usr/bin/llvm-ranlib \
         -D LLVM_BINUTILS_INCDIR=/usr/include \
+        -D LLVM_USE_LINKER=lld \
         -D FFI_INCLUDE_DIR=$(pkg-config --variable=includedir libffi) \
         -D LLVM_VERSION_SUFFIX="" \
         -D LLVM_APPEND_VC_REV=ON \
