diff --git a/PKGBUILD b/PKGBUILD
index 73986b5..f3af78e 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,7 +4,7 @@
 
 pkgname=mate-session-manager
 pkgver=1.26.1
-pkgrel=1
+pkgrel=1.1
 pkgdesc="The MATE Session Handler"
 url="https://mate-desktop.org"
 arch=('x86_64')
@@ -13,11 +13,12 @@ depends=('dbus-glib' 'gettext' 'gtk3' 'libsm' 'mate-desktop')
 makedepends=('xtrans' 'systemd' 'python')
 optdepends=('gnome-keyring: keyring support'
             'xdg-user-dirs-gtk: manage user directories')
-groups=('mate')
+groups=('mate' 'extra-rebuild')
 conflicts=('mate-session-manager-gtk3')
 replaces=('mate-session-manager-gtk3')
 source=("https://pub.mate-desktop.org/releases/${pkgver%.*}/${pkgname}-${pkgver}.tar.xz")
 sha256sums=('5b8c7d6441fd9c293c863882ab67a7493c53cdf64eab27c094575f423ebd4278')
+options=('lto')
 
 build() {
 	cd "${pkgname}-${pkgver}"
