diff --git a/PKGBUILD b/PKGBUILD
index cf0119f..94b5531 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,18 +4,19 @@
 
 pkgname=mate-desktop
 pkgver=1.26.2
-pkgrel=1
+pkgrel=1.1
 pkgdesc="Library with common API for various MATE modules"
 url="http://mate-desktop.org"
 arch=('x86_64')
 license=('GPL' 'LGPL')
-depends=('gtk3' 'startup-notification' 'iso-codes')
+depends=('gtk3' 'startup-notification' 'iso-codes' 'dconf' 'glib2' 'hicolor-icon-theme')
 makedepends=('gobject-introspection' 'intltool')
-groups=('mate')
+groups=('mate' 'extra-rebuild')
 conflicts=('mate-desktop-gtk3' 'mate-desktop-schemas')
 replaces=('mate-desktop-gtk3' 'mate-desktop-schemas')
 source=("https://pub.mate-desktop.org/releases/${pkgver%.*}/${pkgname}-${pkgver}.tar.xz")
 sha256sums=('5ed03f7c52d7b7ece49ca1e736aa766aefeb5c639eab3e68c7090c530d02376a')
+options=('lto')
 
 build() {
 	cd "$pkgname-$pkgver"
@@ -24,7 +25,7 @@ build() {
         	--disable-schemas-compile
 
     	#https://bugzilla.gnome.org/show_bug.cgi?id=656231
-    	sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
+	sed -i -e 's/ -shared / -Wl,-O3,--as-needed\0/g' libtool
 
     	make
 }
