diff --git a/PKGBUILD b/PKGBUILD
index f53b8c9..61a850e 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,7 +4,7 @@
 
 pkgname=mate-panel
 pkgver=1.26.4
-pkgrel=1
+pkgrel=1.1
 pkgdesc="The MATE Panel"
 url="https://mate-desktop.org"
 arch=('x86_64')
@@ -12,10 +12,11 @@ license=('GPL')
 depends=('dbus-glib' 'libwnck3' 'libcanberra' 'libmateweather' 'libsm' 'mate-menus' 'mate-desktop' 'gettext' 'gtk-layer-shell')
 makedepends=('itstool' 'gobject-introspection' 'mate-common' 'yelp-tools' 'autoconf-archive')
 source=("https://pub.mate-desktop.org/releases/${pkgver%.*}/${pkgname}-${pkgver}.tar.xz")
-groups=('mate')
+groups=('mate' 'extra-rebuild')
 conflicts=('mate-panel-gtk3')
 replaces=('mate-panel-gtk3')
 sha256sums=('2070f9d515657e5ddcda0d87ef729713dba9cb7b2ad06223bd674a21cc6b3daf')
+options=('lto')
 
 build() {
     cd "${pkgname}-${pkgver}" 
@@ -27,7 +28,7 @@ build() {
                 --enable-introspection
 
     #https://bugzilla.gnome.org/show_bug.cgi?id=656231
-    sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
+    sed -i -e 's/ -shared / -Wl,-O3,--as-needed\0/g' libtool
 
     make
 }
