diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 9fa1bf4..96e39d5 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -3,7 +3,7 @@
 
 pkgname=mate-panel
 pkgver=1.26.2
-pkgrel=1
+pkgrel=1.1
 pkgdesc="The MATE Panel"
 url="https://mate-desktop.org"
 arch=('x86_64')
@@ -15,6 +15,7 @@ groups=('mate')
 conflicts=('mate-panel-gtk3')
 replaces=('mate-panel-gtk3')
 sha256sums=('aec4f98c89c59e76cc0656ed0482f1d82912f4decc660f21c8d004e493e05410')
+options=('lto')
 
 build() {
     	cd "${pkgname}-${pkgver}" 
