diff --git a/PKGBUILD b/PKGBUILD
index a35dd01..a235468 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -11,13 +11,13 @@ pkgver=4.10.0.0
 pkgrel=1
 pkgdesc='Official MEGA desktop application for syncing with MEGA Cloud Drive'
 arch=('x86_64')
-url='https://github.com/meganz/MEGAsync/'
+url='https://mega.nz/desktop'
 license=('custom')
 depends=('c-ares' 'crypto++' 'curl' 'ffmpeg' 'freeimage' 'icu' 'libmediainfo'
          'libpdfium' 'libsodium' 'libudev.so' 'libuv' 'libxcb' 'libzen' 'openssl'
          'qt5-base' 'qt5-svg' 'qt5-x11extras' 'sqlite' 'zlib'  'hicolor-icon-theme')
 makedepends=('git' 'doxygen' 'lsb-release' 'qt5-tools' 'swig')
-source=("git+https://github.com/meganz/MEGAsync.git#tag=v${pkgver}_OSX"
+source=("git+https://github.com/meganz/MEGAsync.git#tag=v${pkgver}_Linux"
         'meganz-sdk'::'git+https://github.com/meganz/sdk.git'
         '010-megasync-libpdfium.patch'
         '020-megasync-ffmpeg.patch')
