diff --git a/PKGBUILD b/PKGBUILD
index 1ebe9e9..80b1df2 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -13,7 +13,7 @@
 pkgbase=mesa-minimal-git
 pkgname=(mesa-minimal-git opencl-rusticl-mesa-minimal-git)
 pkgdesc="an open-source implementation of the OpenGL specification, stripped down git version"
-pkgver=24.0.0_devel.181605.fde43e44e9f
+pkgver=24.0.0_devel.r181693.g9b2b790ea3c
 pkgrel=1
 arch=('x86_64')
 makedepends=(git meson ninja libglvnd python-mako xorgproto libxml2 libx11  libva elfutils libxrandr
@@ -21,11 +21,14 @@ makedepends=(git meson ninja libglvnd python-mako xorgproto libxml2 libx11  libv
                             rust rust-bindgen spirv-tools-git spirv-llvm-translator-minimal-git libvdpau systemd-libs)
 # In order to keep the package simple and ease troubleshooting only use one llvm implementation
 optdepends=('opengl-man-pages: for the OpenGL API man pages')
-provides=(mesa vulkan-intel vulkan-radeon vulkan-mesa-layer libva-mesa-driver vulkan-swrast mesa-vdpau vulkan-driver opengl-driver)
-conflicts=(mesa vulkan-intel vulkan-radeon vulkan-mesa-layer libva-mesa-driver vulkan-swrast mesa-vdpau)
+provides=(mesa vulkan-mesa-layers vulkan-intel vulkan-radeon vulkan-swrast libva-mesa-driver mesa-vdpau libva-driver mesa-libgl opengl-driver vdpau-driver vulkan-driver)
+# https://gitlab.archlinux.org/archlinux/packaging/packages/mesa/-/blob/60426d25443c61428f1ce77b0cf39cb8e56c646e/PKGBUILD
+conflicts=(mesa vulkan-mesa-layers vulkan-intel vulkan-radeon vulkan-swrast vulkan-virtio libva-mesa-driver mesa-vdpau vulkan-mesa-layer vulkan-mesa mesa-libgl)
+replaces=(vulkan-mesa-layer vulkan-mesa mesa-libgl)
 # mixing components from different mesa versions is a bad idea, conflict with everything unique provided by extra/mesa
 url="https://www.mesa3d.org"
 license=('custom')
+groups=("${pkgbase%%-git}")
 source=("mesa::git+https://gitlab.freedesktop.org/mesa/mesa.git"
                 'LICENSE'
 )
@@ -44,7 +47,7 @@ pkgver() {
     cd mesa
     local _ver
     read -r _ver < VERSION || [ -n "_ver" ]
-    echo ${_ver/-/_}.$(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+    echo ${_ver/-/_}.r$(git rev-list --count HEAD).g$(git rev-parse --short HEAD)
 }
 
 build() {
@@ -124,6 +127,7 @@ package_opencl-rusticl-mesa-minimal-git() {
     pkgdesc="OpenCL support in rust for mesa drivers (git version)"
     conflicts=(opencl-clover-mesa opencl-rusticl-mesa)
     provides=(opencl-rusticl-mesa opencl-driver)
+    replaces=('rusticl-mesa-minimal-git')
     depends=(libdrm spirv-llvm-translator-minimal-git libclc-minimal-git spirv-tools-git
                     mesa-minimal-git=$pkgver-$pkgrel llvm-libs-minimal-git clang-libs-minimal-git
                     expat libelf zstd lm_sensors zlib gcc-libs glibc
