diff --git a/LICENSE b/LICENSE
index ae33d27..060f4a9 100644
--- a/LICENSE
+++ b/LICENSE
@@ -1,33 +1,34 @@
+The Mesa 3D Graphics Library
+
 Disclaimer
 
-Mesa is a 3-D graphics library with an API which is very similar to
-that of OpenGL*
-To the extent that Mesa utilizes the OpenGL command syntax or state
-machine, it is being used with authorization from Silicon Graphics,
-Inc.(SGI). However, the author does not possess an OpenGL license
-from SGI, and makes no claim that Mesa is in any way a compatible
-replacement for OpenGL or associated with SGI. Those who want a
-licensed implementation of OpenGL should contact a licensed
-vendor.
+   Mesa is a 3-D graphics library with an API which is very similar to
+   that of [1]OpenGL.* To the extent that Mesa utilizes the OpenGL command
+   syntax or state machine, it is being used with authorization from
+   [2]Silicon Graphics, Inc.(SGI). However, the author does not possess an
+   OpenGL license from SGI, and makes no claim that Mesa is in any way a
+   compatible replacement for OpenGL or associated with SGI. Those who
+   want a licensed implementation of OpenGL should contact a licensed
+   vendor.
 
-Please do not refer to the library as MesaGL (for legal
-reasons). It's just Mesa or The Mesa 3-D graphics
-library
+   Please do not refer to the library as MesaGL (for legal reasons). It's
+   just Mesa or The Mesa 3-D graphics library.
 
-* OpenGL is a trademark of Silicon Graphics Incorporated.
+   * OpenGL is a trademark of [3]Silicon Graphics Incorporated.
 
 License / Copyright Information
 
-The Mesa distribution consists of several components.  Different copyrights
-and licenses apply to different components.  For example, GLUT is copyrighted
-by Mark Kilgard, some demo programs are copyrighted by SGI, some of the Mesa
-device drivers are copyrighted by their authors.  See below for a list of
-Mesa's main components and the license for each.
+   The Mesa distribution consists of several components. Different
+   copyrights and licenses apply to different components. For example, the
+   GLX client code uses the SGI Free Software License B, and some of the
+   Mesa device drivers are copyrighted by their authors. See below for a
+   list of Mesa's main components and the license for each.
 
-The core Mesa library is licensed according to the terms of the MIT license.
-This allows integration with the XFree86, Xorg and DRI projects.
+   The core Mesa library is licensed according to the terms of the MIT
+   license. This allows integration with the XFree86, Xorg and DRI
+   projects.
 
-The default Mesa license is as follows:
+   The default Mesa license is as follows:
 
 Copyright (C) 1999-2007  Brian Paul   All Rights Reserved.
 
@@ -44,39 +45,39 @@ in all copies or substantial portions of the Software.
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
 OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
-BRIAN PAUL BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
-AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
-CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
 
 Attention, Contributors
 
-When contributing to the Mesa project you must agree to the licensing terms
-of the component to which you're contributing.
-The following section lists the primary components of the Mesa distribution
-and their respective licenses.
-
+   When contributing to the Mesa project you must agree to the licensing
+   terms of the component to which you're contributing. The following
+   section lists the primary components of the Mesa distribution and their
+   respective licenses.
 
 Mesa Component Licenses
 
-Component         Location               Primary Author      License
-----------------------------------------------------------------------------
-Main Mesa code    src/mesa/              Brian Paul          Mesa (MIT)
+Component         Location               License
+------------------------------------------------------------------
+Main Mesa code    src/mesa/              MIT
 
-Device drivers    src/mesa/drivers/*     See drivers         See drivers
+Device drivers    src/mesa/drivers/*     MIT, generally
 
-Ext headers       include/GL/glext.h     SGI                 SGI Free B
-                  include/GL/glxext.h
+Gallium code      src/gallium/           MIT
 
-GLUT              src/glut/              Mark Kilgard        Mark's copyright
-
-Mesa GLU library  src/glu/mesa/          Brian Paul          GNU-LGPL
+Ext headers       include/GL/glext.h     Khronos
+                  include/GL/glxext.h
 
-SGI GLU library   src/glu/sgi/           SGI                 SGI Free B
+GLX client code   src/glx/               SGI Free Software License B
 
-demo programs     progs/demos/           various             see source files
+C11 thread        include/c11/threads*.h Boost (permissive) emulation
 
-X demos           progs/xdemos/          Brian Paul          see source files
+   In general, consult the source files for license terms.
 
-SGI demos         progs/samples/         SGI                 SGI copyright
+References
 
-RedBook demos     progs/redbook/         SGI                 SGI copyright
+   1. https://www.opengl.org/
+   2. https://www.sgi.com/
+   3. https://www.sgi.com/
diff --git a/PKGBUILD b/PKGBUILD
index 1ebe9e9..8bb2026 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -13,7 +13,7 @@
 pkgbase=mesa-minimal-git
 pkgname=(mesa-minimal-git opencl-rusticl-mesa-minimal-git)
 pkgdesc="an open-source implementation of the OpenGL specification, stripped down git version"
-pkgver=24.0.0_devel.181605.fde43e44e9f
+pkgver=24.0.0_rc3.r183197.g808f056688b
 pkgrel=1
 arch=('x86_64')
 makedepends=(git meson ninja libglvnd python-mako xorgproto libxml2 libx11  libva elfutils libxrandr
@@ -21,18 +21,21 @@ makedepends=(git meson ninja libglvnd python-mako xorgproto libxml2 libx11  libv
                             rust rust-bindgen spirv-tools-git spirv-llvm-translator-minimal-git libvdpau systemd-libs)
 # In order to keep the package simple and ease troubleshooting only use one llvm implementation
 optdepends=('opengl-man-pages: for the OpenGL API man pages')
-provides=(mesa vulkan-intel vulkan-radeon vulkan-mesa-layer libva-mesa-driver vulkan-swrast mesa-vdpau vulkan-driver opengl-driver)
-conflicts=(mesa vulkan-intel vulkan-radeon vulkan-mesa-layer libva-mesa-driver vulkan-swrast mesa-vdpau)
+provides=(mesa vulkan-mesa-layers vulkan-intel vulkan-radeon vulkan-swrast libva-mesa-driver mesa-vdpau libva-driver mesa-libgl opengl-driver vdpau-driver vulkan-driver)
+# https://gitlab.archlinux.org/archlinux/packaging/packages/mesa/-/blob/60426d25443c61428f1ce77b0cf39cb8e56c646e/PKGBUILD
+conflicts=(mesa vulkan-mesa-layers vulkan-intel vulkan-radeon vulkan-swrast vulkan-virtio libva-mesa-driver mesa-vdpau vulkan-mesa-layer vulkan-mesa mesa-libgl)
+replaces=(vulkan-mesa-layer vulkan-mesa mesa-libgl)
 # mixing components from different mesa versions is a bad idea, conflict with everything unique provided by extra/mesa
 url="https://www.mesa3d.org"
-license=('custom')
-source=("mesa::git+https://gitlab.freedesktop.org/mesa/mesa.git"
+license=('MIT AND BSD-3-Clause AND SGI-B-2.0')
+groups=("${pkgbase%%-git}")
+source=("mesa::git+https://gitlab.freedesktop.org/mesa/mesa.git#branch=24.0" # amdgpu broken
                 'LICENSE'
 )
 md5sums=('SKIP'
-         '5c65a0fe315dd347e09b1f2826a1df5a')
+         '1eff81373cde4b3d7b0ea7590431c962')
 sha512sums=('SKIP'
-            '25da77914dded10c1f432ebcbf29941124138824ceecaf1367b3deedafaecabc082d463abcfa3d15abff59f177491472b505bcb5ba0c4a51bb6b93b4721a23c2')
+            'f9f0d0ccf166fe6cb684478b6f1e1ab1f2850431c06aa041738563eb1808a004e52cdec823c103c9e180f03ffc083e95974d291353f0220fe52ae6d4897fecc7')
 options=(!emptydirs)
 
 # ninja grabs all available cores and leaves almost nothing for other processes.
@@ -44,7 +47,7 @@ pkgver() {
     cd mesa
     local _ver
     read -r _ver < VERSION || [ -n "_ver" ]
-    echo ${_ver/-/_}.$(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+    echo ${_ver/-/_}.r$(git rev-list --count HEAD).g$(git rev-parse --short HEAD)
 }
 
 build() {
@@ -94,11 +97,29 @@ build() {
 }
 
 package_mesa-minimal-git() {
-    depends=(libdrm libxxf86vm libxdamage libxshmfence libelf
-                        libunwind libglvnd wayland lm_sensors vulkan-icd-loader
-                        zstd llvm-libs-minimal-git zlib expat libxext libxcb systemd-libs
-                        glibc libx11 libxfixes gcc-libs
-                        xcb-util-keysyms
+    depends=(
+        'expat' 'libexpat.so'
+        'gcc-libs'
+        'glibc'
+        'libdrm'
+        'libelf'
+        'libglvnd'
+        'libunwind'
+        'libx11'
+        'libxcb'
+        'libxdamage'
+        'libxext'
+        'libxfixes'
+        'libxshmfence'
+        'libxxf86vm'
+        'llvm-libs-minimal-git'
+        'lm_sensors' 'libsensors.so'
+        'systemd-libs' 'libudev.so'
+        'vulkan-icd-loader'
+        'wayland' 'libwayland-client.so' 'libwayland-server.so'
+        'xcb-util-keysyms'
+        'zlib' 'libz.so'
+        'zstd' 'libzstd.so'
     )
 
     DESTDIR="${pkgdir}" ninja $NINJAFLAGS -C _build install
@@ -124,9 +145,21 @@ package_opencl-rusticl-mesa-minimal-git() {
     pkgdesc="OpenCL support in rust for mesa drivers (git version)"
     conflicts=(opencl-clover-mesa opencl-rusticl-mesa)
     provides=(opencl-rusticl-mesa opencl-driver)
-    depends=(libdrm spirv-llvm-translator-minimal-git libclc-minimal-git spirv-tools-git
-                    mesa-minimal-git=$pkgver-$pkgrel llvm-libs-minimal-git clang-libs-minimal-git
-                    expat libelf zstd lm_sensors zlib gcc-libs glibc
+    replaces=('rusticl-mesa-minimal-git')
+    depends=(
+        "mesa-minimal-git=${pkgver}-${pkgrel}"
+        'clang-libs-minimal-git'
+        'expat' 'libexpat.so'
+        'gcc-libs'
+        'glibc'
+        'libclc-minimal-git'
+        'libdrm'
+        'libelf'
+        'llvm-libs-minimal-git'
+        'spirv-llvm-translator-minimal-git'
+        'spirv-tools-git'
+        'zlib' 'libz.so'
+        'zstd' 'libzstd.so'
     )
     
     
