diff --git a/PKGBUILD b/PKGBUILD
index d413b07..56ee2bc 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -21,11 +21,14 @@ makedepends=(git meson ninja libglvnd python-mako xorgproto libxml2 libx11  libv
                             rust rust-bindgen spirv-tools-git spirv-llvm-translator-minimal-git libvdpau systemd-libs)
 # In order to keep the package simple and ease troubleshooting only use one llvm implementation
 optdepends=('opengl-man-pages: for the OpenGL API man pages')
-provides=(mesa vulkan-intel vulkan-radeon vulkan-mesa-layer libva-mesa-driver vulkan-swrast mesa-vdpau vulkan-driver opengl-driver)
-conflicts=(mesa vulkan-intel vulkan-radeon vulkan-mesa-layer libva-mesa-driver vulkan-swrast mesa-vdpau)
+provides=(mesa vulkan-mesa-layers vulkan-intel vulkan-radeon vulkan-swrast libva-mesa-driver mesa-vdpau libva-driver mesa-libgl opengl-driver vdpau-driver vulkan-driver)
+# https://gitlab.archlinux.org/archlinux/packaging/packages/mesa/-/blob/60426d25443c61428f1ce77b0cf39cb8e56c646e/PKGBUILD
+conflicts=(mesa vulkan-mesa-layers vulkan-intel vulkan-radeon vulkan-swrast vulkan-virtio libva-mesa-driver mesa-vdpau vulkan-mesa-layer vulkan-mesa mesa-libgl)
+replaces=(vulkan-mesa-layer vulkan-mesa mesa-libgl)
 # mixing components from different mesa versions is a bad idea, conflict with everything unique provided by extra/mesa
 url="https://www.mesa3d.org"
 license=('custom')
+groups=("${pkgbase%%-git}")
 source=("mesa::git+https://gitlab.freedesktop.org/mesa/mesa.git"
                 'LICENSE'
 )
@@ -124,6 +127,7 @@ package_opencl-rusticl-mesa-minimal-git() {
     pkgdesc="OpenCL support in rust for mesa drivers (git version)"
     conflicts=(opencl-clover-mesa opencl-rusticl-mesa)
     provides=(opencl-rusticl-mesa opencl-driver)
+    replaces=('rusticl-mesa-minimal-git')
     depends=(libdrm spirv-llvm-translator-minimal-git libclc-minimal-git spirv-tools-git
                     mesa-minimal-git=$pkgver-$pkgrel llvm-libs-minimal-git clang-libs-minimal-git
                     expat libelf zstd lm_sensors zlib gcc-libs glibc
