diff --git a/PKGBUILD b/PKGBUILD
index 52f74f1..25b07ce 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,8 +3,8 @@
 _pkgbase=moe-theme
 pkgbase="${_pkgbase}-git"
 pkgname=(plasma-theme-moe-git kvantum-theme-moe-git)
-pkgver=r29.bd9b178
-pkgrel=1
+pkgver=r41.0eae9f1
+pkgrel=2
 pkgdesc='Moe is a light theme, with blur transparencies and red tones with gradients.'
 arch=('any')
 url="https://gitlab.com/jomada/${_pkgbase}"
@@ -43,7 +43,7 @@ package_plasma-theme-moe-git() {
 }
 
 package_kvantum-theme-moe-git() {
-    depends=(kvantum-qt5)
+    depends=('kvantum')
 
     cd "${srcdir}/${_pkgbase}"
 
