diff --git a/PKGBUILD b/PKGBUILD
index cbc1b3b..f2042f7 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,11 +4,11 @@ _pkgbase=moe-theme
 pkgbase="${_pkgbase}-git"
 pkgname=(plasma5-themes-moe-git kvantum-theme-moe-git)
 pkgver=r41.0eae9f1
-pkgrel=1
+pkgrel=3
 pkgdesc='A light theme, with blur transparencies and red tones with gradients.'
 arch=('any')
 url="https://gitlab.com/jomada/${_pkgbase}"
-license=('GPL3')
+license=('GPL-3.0-or-later')
 makedepends=('git')
 options=(!strip)
 source=("${_pkgbase}::git+https://gitlab.com/jomada/${_pkgbase}")
@@ -20,7 +20,9 @@ pkgver() {
 }
 
 package_plasma5-themes-moe-git() {
-    optdepends=('kvantum-theme-moe-git: Moe theme for Kvantum (recommended)')
+    provides=("${pkgname%-git}")
+    replaces=('plasma-theme-moe-git')
+    conflicts=('plasma-theme-moe-git')
 
     cd "${_pkgbase}"
 
@@ -39,6 +41,7 @@ package_plasma5-themes-moe-git() {
 
 package_kvantum-theme-moe-git() {
     depends=(kvantum)
+    provides=("${pkgname%-git}")
 
     cd "${_pkgbase}"
 
