diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 14f9358..532c145 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -2,12 +2,13 @@
 # Contributor: Bartłomiej Piotrowski <bpiotrowski@archlinux.org>
 # Contributor: Eivind Uggedal <eivind@uggedal.com>
 
-pkgname=mpv
+_pkgname=mpv
+pkgname=$_pkgname-vapoursynth
 epoch=1
 _tag='33f10b1aee5fef45960728fe8595d1603c8a9a85' # git rev-parse v${pkgver}
 pkgver=0.35.0
-pkgrel=3
-pkgdesc='a free, open source, and cross-platform media player'
+pkgrel=2
+pkgdesc='a free, open source, and cross-platform media player (with Vapoursynth libs)'
 arch=('x86_64')
 # We link against libraries that are licensed GPLv3 explicitly, so our
 # package is GPLv3 only as well. (Is this still correct?)
@@ -23,19 +24,20 @@ depends=('alsa-lib' 'libasound.so' 'desktop-file-utils' 'ffmpeg' 'libavcodec.so'
          'libxext' 'libxinerama' 'libxkbcommon' 'libxkbcommon.so' 'libxpresent' 'libxrandr' 'libxss'
          'libxv' 'luajit' 'mesa' 'mujs' 'pipewire' 'rubberband' 'librubberband.so' 'shaderc'
          'libshaderc_shared.so' 'uchardet' 'vulkan-icd-loader' 'wayland' 'xdg-utils'
-         'zlib')
+         'zlib' 'vapoursynth')
 makedepends=('git' 'python-docutils' 'ladspa' 'wayland-protocols'
              'ffnvcodec-headers' 'vulkan-headers' 'waf')
 optdepends=('yt-dlp: for video-sharing websites playback'
             'youtube-dl: for video-sharing websites playback')
-provides=('libmpv.so')
+provides=('libmpv.so' 'mpv')
+conflicts=('mpv')
 options=('debug' '!emptydirs')
 validpgpkeys=('145077D82501AA20152CACCE8D769208D5E31419') # sfan5 <sfan5@live.de>
 source=("git+https://github.com/mpv-player/mpv.git#tag=${_tag}?signed")
 sha256sums=('SKIP')
 
 build() {
-  cd ${pkgname}
+  cd ${_pkgname}
 
   waf configure --prefix=/usr \
     --confdir=/etc/mpv \
@@ -45,13 +47,14 @@ build() {
     --enable-libarchive \
     --enable-libmpv-shared \
     --disable-build-date \
-    --disable-caca
+    --disable-caca \
+    --enable-vapoursynth
 
   waf build
 }
 
 package() {
-  cd ${pkgname}
+  cd ${_pkgname}
 
   waf install --destdir="$pkgdir"
 
