diff --git a/PKGBUILD b/PKGBUILD
index 7e32dc6..11d0c1b 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -5,8 +5,8 @@
 # Contributor: Emīls Piņķis <emil at mullvad dot net>
 # Contributor: Andrej Mihajlov <and at mullvad dot net>
 pkgname=mullvad-vpn-beta
-_pkgver=2022.4
-_channel=stable
+_pkgver=2022.5
+_channel=beta1
 _rel=1
 # beta
 #pkgver=${_pkgver}.${_channel}${_rel}
@@ -18,13 +18,13 @@ arch=('x86_64')
 url="https://www.mullvad.net"
 license=('GPL3')
 depends=('iputils' 'libnotify' 'libappindicator-gtk3' 'nss')
-makedepends=('cargo' 'git' 'go' 'nodejs>=16' 'npm>=8.3' 'libxcrypt-compat')
+makedepends=('cargo' 'git' 'go' 'nodejs>=16' 'npm>=8.3' 'libxcrypt-compat' 'vim')
 provides=("${pkgname%-beta}")
 conflicts=("${pkgname%-beta}")
 options=('!lto')
 install="${pkgname%-beta}.install"
-_tag=06c847a7a7d72a148bd5dba0a44860eb1b3aa182 # tags/2022.4^0
-_commit=b63c5c8c7977963aeb585b6ddd4537dffe2aeeec
+_tag=51c3edc934ca0ac77cfb348871a14755b9f52e6d # tags/2022.5-beta1^0
+_commit=f6dca66645c82501a330416ad39c7e63bcdae57d
 source=("git+https://github.com/mullvad/mullvadvpn-app.git#commit=${_tag}?signed"
         "git+https://github.com/mullvad/mullvadvpn-app-binaries.git#commit=${_commit}?signed"
         "${pkgname%-beta}.sh")
@@ -42,7 +42,11 @@ prepare() {
   git submodule update
 
   # Disable building of rpm
-  sed -i "s/'deb', 'rpm'/'deb'/g" gui/tasks/distribution.js
+  # FIXME: Do we really need to use vim to do this?
+  vim -e - gui/tasks/distribution.js << @@@
+g/'rpm'/.-1,.2d
+wq
+@@@
 
   export RUSTUP_TOOLCHAIN=stable
   echo "Removing old Rust build artifacts"
@@ -144,13 +148,12 @@ package() {
   install -d "$pkgdir/opt/Mullvad VPN"
   cp -r dist/linux-unpacked/* "$pkgdir/opt/Mullvad VPN/"
 
-  # Symlink daemon service to correct directory
-  install -d "$pkgdir/usr/lib/systemd/system"
-  ln -s "/opt/Mullvad VPN/resources/mullvad-daemon.service" \
-    "$pkgdir/usr/lib/systemd/system/"
+  # Install daemon services
+  install -Dm644 dist-assets/linux/mullvad-{daemon,early-boot-blocking}.service \
+    -t "$pkgdir/usr/lib/systemd/system/"
 
   # Install binaries
-  install -Dm755 dist-assets/{mullvad,mullvad-exclude} -t "$pkgdir/usr/bin/"
+  install -Dm755 dist-assets/mullvad{,-daemon,-exclude} -t "$pkgdir/usr/bin/"
 
   # Link to the problem report binary
   ln -s "/opt/Mullvad VPN/resources/mullvad-problem-report" \
diff --git a/mullvad-vpn.install b/mullvad-vpn.install
index a03c95f..ea824c8 100644
--- a/mullvad-vpn.install
+++ b/mullvad-vpn.install
@@ -7,6 +7,8 @@ pre_install() {
       || echo "Failed to copy old daemon log"
   fi
 
+  systemctl disable mullvad-early-boot-blocking || true
+
   pkill -x "mullvad-gui" || true
 
   rm -f /var/cache/mullvad-vpn/relays.json
@@ -17,6 +19,7 @@ post_install() {
   chmod u+s "/usr/bin/mullvad-exclude"
 
   echo 'Enabling Mullvad VPN daemon...'
+  systemctl enable --now mullvad-early-boot-blocking
   systemctl enable --now mullvad-daemon
 }
 
@@ -27,6 +30,7 @@ pre_upgrade() {
 post_upgrade() {
   echo 'Starting & enabling Mullvad VPN daemon...'
   systemctl daemon-reload
+  systemctl enable --now mullvad-early-boot-blocking
   systemctl enable --now mullvad-daemon
 
   chmod u+s "/usr/bin/mullvad-exclude"
@@ -37,6 +41,7 @@ pre_remove() {
   echo 'Making sure the Mullvad VPN daemon is stopped & disabled...'
   systemctl stop mullvad-daemon || true
   systemctl disable mullvad-daemon || true
+  systemctl disable mullvad-early-boot-blocking || true
 
   pkill -x "mullvad-gui" || true
 
