diff --git a/PKGBUILD b/PKGBUILD
index 38e9efe..0e19eac 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,18 +1,16 @@
 # Maintainer: Frederic Bezies <fredbezies at gmail dot com>
 # Contributor: Morten Linderud <morten@linderud.pw> 
 pkgname=nerd-fonts-git
-pkgver=2.3.1.r1.g50a189317
+pkgver=2.3.3.r38.gbde5c7def
 pkgrel=1
 epoch=1
 pkgdesc="Iconic font aggregator, collection, and patcher: 34 patched fonts (77k variations). Includes popular glyph collections such as Font Awesome & fonts such as Hack"
 _gitname=nerd-fonts
 arch=('any')
 url='https://github.com/ryanoasis/nerd-fonts'
-license=('MIT')
-conflicts=('nerd-fonts' 'ttf-iosevka-nerd')
+license=('custom:MIT')
 makedepends=('git' 'fontconfig' 'xorg-mkfontscale' 'xorg-bdftopcf' 'xorg-font-util')
 conflicts=(
-    'nerd-fonts'
     'nerd-fonts-complete'
     'nerd-fonts-complete-mono-glyphs'
     'nerd-fonts-anonymous-pro'
@@ -70,8 +68,16 @@ package() {
   install -dm755 "${ttfdir}"
 
   # Install fonts
-  find "${srcdir}/${_gitname}/patched-fonts" -iname "*.ttf" -execdir install -m644 {} "${ttfdir}/{}" \;
-  find "${srcdir}/${_gitname}/patched-fonts" -iname "*.otf" -execdir install -m644 {} "${otfdir}/{}" \;
+  # Intervention: Avoid installing Windows-compatibile fonts for obvious reasons, and symbols-only
+  # fonts for compatibility reasons because don't want to replace official packages in any capacity
+  find "${srcdir}/${_gitname}/patched-fonts" \
+    -not \( -path "${srcdir}/${_gitname}/patched-fonts/NerdFontsSymbolsOnly" -prune \) \
+    -not \( -iname "*Windows Compatible*" -prune \) -iname "*.ttf" \
+    -execdir install -m644 {} "${ttfdir}/{}" \;
+  find "${srcdir}/${_gitname}/patched-fonts" \
+    -not \( -path "${srcdir}/${_gitname}/patched-fonts/NerdFontsSymbolsOnly" -prune \) \
+    -not \( -iname "*Windows Compatible*" -prune \) -iname "*.otf" \
+    -execdir install -m644 {} "${otfdir}/{}" \;
 
   # Install scripts
   find "${srcdir}/${_gitname}/bin/scripts/lib" -iname "*.sh" -execdir install -m644 {} "${libdir}/{}" \;
