diff --git a/PKGBUILD b/PKGBUILD
index 1385e7a..129812b 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,19 +2,23 @@
 # Maintainer: Primalmotion <primalmotion@pm.me>
 pkgname=networkd-dispatcher
 pkgver=2.2.4
-pkgrel=0
+pkgrel=2
 pkgdesc="A dispatcher daemon for systemd-networkd events"
 arch=('any')
 url="https://gitlab.com/craftyguy/networkd-dispatcher"
-license=('GPL3')
+license=('GPL-3.0-or-later')
 makedepends=('asciidoc')
 depends=('python' 'python-dbus' 'python-gobject' 'dbus-glib')
 optdepends=('wireless_tools: for interacting with wlan interfaces', 'iw: for interacting with wlan interfaces')
-provides=("${pkgname}")
-conflicts=("${pkgname}")
 backup=('etc/conf.d/networkd-dispatcher.conf')
-source=("https://gitlab.com/craftyguy/${pkgname}/-/archive/${pkgver}/${pkgname}-${pkgver}.tar.bz2")
-sha512sums=('6eb8a3d2d2c4b1c57b10e6b7539744c6747e5a0e3680937190d3a802fab55c491a3560727a7f84354be71dc8bf50a7432deaa624a0bdd6671ebc9cd655f1e564')
+source=("https://gitlab.com/craftyguy/${pkgname}/-/archive/${pkgver}/${pkgname}-${pkgver}.tar.bz2"
+        "${pkgname}-fix-notification-access.patch::https://gitlab.com/craftyguy/${pkgname}/-/commit/4796368d88da516fafda321d8565ae8ccf465120.patch")
+sha512sums=('6eb8a3d2d2c4b1c57b10e6b7539744c6747e5a0e3680937190d3a802fab55c491a3560727a7f84354be71dc8bf50a7432deaa624a0bdd6671ebc9cd655f1e564'
+            'a83140acc234673f463e539432e10b88b1cfe72925b0130cfe62d0397bbb1b1013531cb36c5e0aceae7e1ee9cb877455be8374a9fe348500172a285f9f15fde1')
+
+prepare() {
+        patch -d "${pkgname}-${pkgver}" -N -p1 <"${pkgname}-fix-notification-access.patch"
+}
 
 package() {
         cd "${pkgname}-${pkgver}"
