diff --git a/PKGBUILD b/PKGBUILD
index 27d3f48..976529d 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,44 +1,54 @@
 # Maintainer: ChinaNuke <chinanuke@nuke666.cn>
 # Contributor: Shayne Hartford <shayneehartford@gmail.com>
 
-pkgname=plasma5-themes-layan-git
-pkgver=r75.c4cce9d
+pkgbase=plasma5-themes-layan-git
+pkgname=('plasma5-themes-layan-git' 'sddm-theme-layan-git' 'kvantum-theme-layan-git')
+pkgver=r75.gc4cce9d
 pkgrel=1
 pkgdesc="A flat Design theme for KDE Plasma desktop"
 arch=(any)
 url="https://github.com/vinceliuice/Layan-kde"
-license=('GPL3')
+license=('GPL-3.0-only')
 makedepends=('git')
-optdepends=('kvantum: Better looking for application styles'
-            'tela-icon-theme: Matching icon theme')
-provides=('layan-kde-git')
-replaces=('layan-kde-git')
-conflicts=('layan-kde-git')
 options=('!strip')
-source=("${pkgname%-git}::git+https://github.com/vinceliuice/Layan-kde.git")
+source=("${pkgbase%-git}::git+https://github.com/vinceliuice/Layan-kde.git")
 sha256sums=('SKIP')
 
 pkgver() {
-    cd "$srcdir/${pkgname%-git}"
-    printf "r%s.%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
+    cd "$srcdir/${pkgbase%-git}"
+    printf "r%s.g%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
 }
 
-prepare() {
-    cd "$srcdir/${pkgname%-git}"
-    sed -i "s#/usr/share#$pkgdir/usr/share#" install.sh
-    cd "$srcdir/${pkgname%-git}/sddm"
-    sed -i "s#/usr/share#$pkgdir/usr/share#" install.sh
-}
+package_plasma5-themes-layan-git() {
+    optdepends=('tela-icon-theme: Matching icon theme')
+    provides=("${pkgname%-git}")
+    replaces=('layan-kde-git')
+    conflicts=('layan-kde-git')
 
-package() {
     # Install aurorae themes, kvantum themes, plasma color schemes, 
     # plasma desktop themes, plasma look-and-feel packages.
-    cd "$srcdir/${pkgname%-git}"
+    cd "$srcdir/${pkgbase%-git}"
     install -d "$pkgdir/usr/share"
-    ./install.sh
+    sed "s#/usr/share#$pkgdir/usr/share#" install.sh | bash -
+
+    # Kvantum themes should belong to separate package
+    mv "$pkgdir/usr/share/Kvantum" "$srcdir/"
+}
+
+package_sddm-theme-layan-git() {
+    depends=('sddm')
+    provides=("${pkgname%-git}")
 
     # Install sddm themes.
-    cd "$srcdir/${pkgname%-git}/sddm"
+    cd "$srcdir/${pkgbase%-git}/sddm"
     install -d "$pkgdir/usr/share/sddm/themes"
-    ./install.sh
+    sed "s#/usr/share#$pkgdir/usr/share#" install.sh | bash -
+}
+
+package_kvantum-theme-layan-git() {
+    depends=('kvantum')
+    provides=("${pkgname%-git}")
+
+    install -d "$pkgdir/usr/share"
+    mv "$srcdir/Kvantum" "$pkgdir/usr/share/"
 }
