diff --git a/PKGBUILD b/PKGBUILD
index 3fe380d..8a3c5d8 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -6,8 +6,8 @@ pkgdesc="Install and manage Proton-GE and Luxtorpeda for Steam and Wine-GE for L
 arch=('any')
 url="https://davidotek.github.io/protonup-qt"
 license=('GPL3')
-depends=('pyside6' 'python-inputs' 'python-psutil' 'python-requests'
-         'python-steam' 'python-vdf' 'python-pyxdg' 'python-pyaml' 'qt6-tools')
+depends=('pyside6' 'python-inputs' 'python-psutil'
+         'python-steam' 'python-pyxdg' 'python-pyaml' 'qt6-tools')
 makedepends=('python-build' 'python-installer' 'python-setuptools' 'python-wheel')
 source=("$pkgname-$pkgver.tar.gz::https://github.com/DavidoTek/ProtonUp-Qt/archive/refs/tags/v$pkgver.tar.gz"
         'net.davidotek.pupgui2.sh')
