diff --git a/PKGBUILD b/PKGBUILD
index d37cd00..6de1a4f 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,7 +9,7 @@ pkgrel=1
 pkgdesc='A library to manipulate gettext files'
 url='https://pypi.python.org/pypi/polib'
 arch=('any')
-license=('MIT')
+license=('custom:MIT')
 depends=('python')
 makedepends=('python-setuptools')
 source=("${pkgbase}-${pkgver}.tar.gz::https://pypi.io/packages/source/p/${_pkgname}/${_pkgname}-${pkgver}.tar.gz")
