diff --git a/PKGBUILD b/PKGBUILD
index 3affd49..0e5a549 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -6,7 +6,7 @@ pkgrel=1
 pkgdesc="Python package for interacting with Steam"
 arch=('any')
 url="https://github.com/ValvePython/steam"
-license=('MIT')
+license=('custom:MIT')
 depends=('python' 'python-cachetools' 'python-pycryptodomex' 'python-requests'
          'python-six' 'python-vdf')
 makedepends=('python-build' 'python-installer' 'python-setuptools' 'python-wheel')
