diff --git a/PKGBUILD b/PKGBUILD
index d37cd00..d9ceb1f 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,16 +4,16 @@
 pkgbase=python-polib
 pkgname=('python-polib')
 _pkgname=polib
-pkgver=1.1.1
+pkgver=1.2.0
 pkgrel=1
 pkgdesc='A library to manipulate gettext files'
 url='https://pypi.python.org/pypi/polib'
 arch=('any')
-license=('MIT')
+license=('custom:MIT')
 depends=('python')
 makedepends=('python-setuptools')
 source=("${pkgbase}-${pkgver}.tar.gz::https://pypi.io/packages/source/p/${_pkgname}/${_pkgname}-${pkgver}.tar.gz")
-sha256sums=('e02c355ae5e054912e3b0d16febc56510eff7e49d60bf22aecb463bd2f2a2dfa')
+sha256sums=('f3ef94aefed6e183e342a8a269ae1fc4742ba193186ad76f175938621dbfc26b')
 
 build() {
   cd "${srcdir}/${_pkgname}-${pkgver}"
