diff --git a/PKGBUILD b/PKGBUILD
index 47103f0..83de530 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,7 +8,7 @@ pkgrel=1
 pkgdesc="Sphinx extension for generating API documentation."
 arch=('any')
 url="https://sphinx-automodapi.readthedocs.io"
-license=('BSD')
+license=('BSD-3-Clause')
 makedepends=('python-setuptools-scm'
              'python-wheel'
              'python-build'
@@ -43,7 +43,7 @@ check() {
 }
 
 package_python-sphinx-automodapi() {
-    depends=('python-sphinx>=4')
+    depends=('python' 'python-sphinx>=4')
     optdepends=('python-sphinx_rtd_theme: rtd'
                 'python-sphinx-automodapi-doc: Documentation for sphinx-automodapi')
     cd ${srcdir}/${_pyname}-${pkgver}
