diff --git a/PKGBUILD b/PKGBUILD
index 851579d..258e497 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,7 +8,7 @@ pkgver=9.012.04
 pkgrel=1
 url="https://www.realtek.com/en/component/zoo/category/network-interface-controllers-10-100-1000m-gigabit-ethernet-pci-express-software"
 pkgdesc="Kernel module for RTL8125"
-license=('GPL2')
+license=('GPL-2.0-or-later')
 arch=('any')
 depends=('dkms')
 conflicts=("${_pkgname}")
