diff --git a/PKGBUILD b/PKGBUILD
index 198c6b4..ec4ee70 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,19 +8,25 @@
 _pkgbase=r8168
 pkgname=${_pkgbase}-dkms
 pkgver=8.052.01
-pkgrel=1
+pkgrel=3
 pkgdesc="A kernel module for Realtek 8168 network cards (DKMS version)"
 url="https://github.com/mtorromeo/r8168"
-license=("GPL")
-arch=('i686' 'x86_64')
-depends=('glibc' 'dkms')
+license=('GPL-2.0-or-later')
+arch=('any')
+depends=('dkms')
 makedepends=('git')
 conflicts=("${_pkgbase}")
 provides=("${_pkgbase}")
 source=("r8168-dkms::git+https://github.com/mtorromeo/r8168.git#tag=${pkgver}"
+        'r8168-dkms-6.8.patch::https://patch-diff.githubusercontent.com/raw/mtorromeo/r8168/pull/57.patch'
         'dkms.conf')
-sha256sums=('SKIP'
-            'd37b8acbd4fe06f81538581712a04751a96fc37bad3a4bd3ae8329f8744c49b3')
+sha256sums=('2c6861ddf87c1ebd40c57a78df2775e8131dc4e7cb57aa9ee22197e39198626d'
+            'a834e3e43745a737ca211c3eb550d369bf1894211e7584717f87b781246fb2d1'
+            'dcca7e1c6c5c7ab629af508813269e1327bf0e51d6a262f1942f247eb57ab402')
+
+prepare() {
+	patch -d r8168-dkms -N -p1 <r8168-dkms-6.8.patch
+}
 
 package() {
 	install -Dm644 dkms.conf "${pkgdir}/usr/src/${_pkgbase}-${pkgver}/dkms.conf"
@@ -31,7 +37,4 @@ package() {
 
 	cd "${pkgname}"
 	cp -dr --no-preserve='ownership' src "${pkgdir}/usr/src/${_pkgbase}-${pkgver}/src"
-
-	echo "blacklist r8169" | \
-		install -Dm644 /dev/stdin "$pkgdir/usr/lib/modprobe.d/$pkgname.conf"
 }
diff --git a/dkms.conf b/dkms.conf
index 6cf4366..7d2eed1 100644
--- a/dkms.conf
+++ b/dkms.conf
@@ -1,6 +1,6 @@
 PACKAGE_NAME="@PKGNAME@"
 PACKAGE_VERSION="@PKGVER@"
-MAKE[0]="make -C $kernel_source_dir M=$dkms_tree/$module/$module_version/build/src EXTRA_CFLAGS='-DCONFIG_R8168_NAPI=y -DCONFIG_R8168_VLAN=y -DCONFIG_ASPM=y -DENABLE_S5WOL=y -DENABLE_EEE=y' modules"
+MAKE[0]="make -C $kernel_source_dir M=$dkms_tree/$module/$module_version/build/src EXTRA_CFLAGS='-DCONFIG_R8168_VLAN=y -DCONFIG_ASPM=y -DENABLE_S5WOL=y -DENABLE_EEE=y -DCONFIG_DYNAMIC_ASPM=y' modules"
 CLEAN="rm src/@PKGNAME@.ko src/*.o || true"
 BUILT_MODULE_NAME[0]="@PKGNAME@"
 BUILT_MODULE_LOCATION[0]="src/"
