diff --git a/PKGBUILD b/PKGBUILD
index 2d893f7..e03c448 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -5,11 +5,11 @@ _pkgbase="sddm"
 pkgname="$_pkgbase-git"
 pkgver=0.19.0.170.g3e48649
 pkgrel=1
-pkgdesc="The Simple Desktop Display Manager"
+pkgdesc="QML based X11 and Wayland display manager (git version)"
 arch=("x86_64")
 url="https://github.com/sddm/sddm"
 license=("GPL")
-depends=('qt5-declarative' 'xorg-xauth' 'xorg-server')
+depends=('qt5-declarative' 'xorg-xauth' 'xorg-server' 'ttf-font')
 makedepends=('extra-cmake-modules' 'python-docutils' 'qt5-tools' 'git')
 provides=("$_pkgbase" 'display-manager')
 conflicts=("$_pkgbase")
