diff --git a/PKGBUILD b/PKGBUILD
index 75dc923..405201c 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,20 +8,20 @@ pkgdesc="Service and tools for management of snap packages."
 depends=('squashfs-tools' 'libseccomp' 'libsystemd' 'apparmor')
 optdepends=('bash-completion: bash completion support'
             'xdg-desktop-portal: desktop integration')
-pkgver=2.60
+pkgver=2.60.2
 pkgrel=1
 arch=('x86_64' 'i686' 'armv7h' 'aarch64')
 url="https://github.com/snapcore/snapd"
 license=('GPL3')
-makedepends=('git' 'go' 'go-tools' 'libseccomp' 'libcap' 'systemd' 'xfsprogs' 'python-docutils' 'apparmor')
+makedepends=('git' 'go' 'go-tools' 'libcap' 'systemd' 'xfsprogs' 'python-docutils')
 conflicts=('snap-confine')
 options=('!strip' 'emptydirs' '!lto')
 install=snapd.install
 source=(
-    "$pkgname-$pkgver.tar.xz::https://github.com/snapcore/${pkgname}/releases/download/${pkgver}/${pkgname}_${pkgver}.vendor.tar.xz"
+    "$pkgname-$pkgver.tar.xz::https://github.com/snapcore/${pkgname}/releases/download/${pkgver}/${pkgname}_${pkgver}.tar.xz"
 )
-sha256sums=(
-    '1d940ba7cee1b6a1fd961c591abf8fdde9d313eff0fb8b6967ad01f40426f33e'
+b2sums=(
+    'a8b49dfb23ddfb60e8037093f69825c35ec8c4278004d159d9b2e9127c15038e8eacd54c58b5c5a29d3b049a18307a38ace025ad6b149da042ee285276bda38a'
 )
 
 
