diff --git a/PKGBUILD b/PKGBUILD
index 130cf03..ace74e9 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,19 +8,19 @@ pkgdesc="Service and tools for management of snap packages."
 depends=('squashfs-tools' 'libseccomp' 'libsystemd' 'apparmor')
 optdepends=('bash-completion: bash completion support'
             'xdg-desktop-portal: desktop integration')
-pkgver=2.57.1
-pkgrel=2
+pkgver=2.57.2
+pkgrel=1
 arch=('x86_64' 'i686' 'armv7h' 'aarch64')
 url="https://github.com/snapcore/snapd"
 license=('GPL3')
-makedepends=('git' 'go' 'go-tools' 'libseccomp' 'libcap' 'systemd' 'xfsprogs' 'python-docutils' 'apparmor')
+makedepends=('git' 'go' 'go-tools' 'libcap' 'systemd' 'xfsprogs' 'python-docutils')
 conflicts=('snap-confine')
 options=('!strip' 'emptydirs' '!lto')
 install=snapd.install
 source=(
     "$pkgname-$pkgver.tar.xz::https://github.com/snapcore/${pkgname}/releases/download/${pkgver}/${pkgname}_${pkgver}.vendor.tar.xz"
 )
-sha256sums=('01a421514ced3d7856ce4128b3812d1ca83cdc8f8f438037a1ffd857ac78e4c8')
+sha256sums=('e4097e1eb8b4845cdebdc08d3861c63a7154b0494fad96569cf7847c6088b2ec')
 
 
 _gourl=github.com/snapcore/snapd
