diff --git a/PKGBUILD b/PKGBUILD
index 75dc923..4dbbdca 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,11 +9,11 @@ depends=('squashfs-tools' 'libseccomp' 'libsystemd' 'apparmor')
 optdepends=('bash-completion: bash completion support'
             'xdg-desktop-portal: desktop integration')
 pkgver=2.60
-pkgrel=1
+pkgrel=2
 arch=('x86_64' 'i686' 'armv7h' 'aarch64')
 url="https://github.com/snapcore/snapd"
 license=('GPL3')
-makedepends=('git' 'go' 'go-tools' 'libseccomp' 'libcap' 'systemd' 'xfsprogs' 'python-docutils' 'apparmor')
+makedepends=('git' 'go' 'go-tools' 'libcap' 'systemd' 'xfsprogs' 'python-docutils')
 conflicts=('snap-confine')
 options=('!strip' 'emptydirs' '!lto')
 install=snapd.install
