diff --git a/PKGBUILD b/PKGBUILD
index b3deefa..ba83e48 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,14 +2,14 @@
 # Contributor: Eric Engestrom <aur [at] engestrom [dot] ch>
 
 pkgname=spirv-tools-git
-pkgver=r3673.e0936b6
+pkgver=1.3.250.0.r92.ge68fe9be4
 pkgrel=1
 epoch=1
 pkgdesc='API and commands for processing SPIR-V modules'
 url='https://github.com/KhronosGroup/SPIRV-Tools'
 arch=('i686' 'x86_64')
 license=('custom')
-groups=('vulkan-devel')
+groups=('vulkan-devel' 'mesa-minimal')
 source=('git+https://github.com/KhronosGroup/SPIRV-Tools'
               'git+https://github.com/google/googletest.git'
               'git+https://github.com/google/effcee.git'
@@ -37,7 +37,7 @@ prepare() {
 
 pkgver() {
   cd SPIRV-Tools
-  printf "r%s.%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short=7 HEAD)"
+  git describe --long --tags --match 'sdk-*' | sed 's/^sdk-//; s/\([^-]*-g\)/r\1/; s/-/./g'
 }
 
 
