diff --git a/PKGBUILD b/PKGBUILD
index 2b215b1..7f97f59 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,14 +4,16 @@ _srcname=SPIRV-LLVM-Translator
 pkgname=spirv-llvm-translator-minimal-git
 pkgdesc="Tool and a library for bi-directional translation between SPIR-V and LLVM IR, trunk version"
 epoch=1
-pkgver=r1817.066514b
-pkgrel=2
+pkgver=18.0.0_r1874.89ffe391907a
+pkgrel=1
 arch=('x86_64')
 url="https://github.com/KhronosGroup/SPIRV-LLVM-Translator"
 license=('custom:Apache 2.0 with LLVM Exception')
-makedepends=(llvm-minimal-git git cmake spirv-headers-git spirv-tools-git)
-checkdepends=(python python-setuptools clang-minimal-git)
-depends=(llvm-libs-minimal-git spirv-tools-git gcc-libs glibc)
+groups=('mesa-minimal')
+# edit makedepends array so we build with our llvm-git
+makedepends=(llvm-git git cmake spirv-headers-git spirv-tools-git)
+checkdepends=(python python-setuptools)
+depends=(spirv-tools-git gcc-libs glibc)
 provides=(spirv-llvm-translator)
 conflicts=(spirv-llvm-translator)
 source=("git+$url.git"
@@ -21,9 +23,16 @@ md5sums=('SKIP')
 sha512sums=('SKIP')
 
 
+#prepare() {
+#    sed 's/[0-9][0-9].0.0/18.0.0/' -i "${_srcname}/CMakeLists.txt"
+#}
+
 pkgver() {
     cd $_srcname
-    printf "r%s.%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short=7 HEAD)"
+    printf "%s_r%s.%s" \
+        "$(grep \(BASE_LLVM_VERSION CMakeLists.txt | awk '{print $3}' | sed 's/[)]//')" \
+        "$(git rev-list --count HEAD)" \
+        "$(git rev-parse --short=12 HEAD)"
 }
 
 build() {
@@ -32,13 +41,19 @@ build() {
         -B _build \
         -S "$srcdir"/$_srcname  \
         -G "Unix Makefiles" \
+        -D CMAKE_AR=/usr/bin/llvm-ar \
         -D CMAKE_BUILD_TYPE=Release \
+        -D CMAKE_C_COMPILER=clang \
+        -D CMAKE_CXX_COMPILER=clang++ \
         -D CMAKE_INSTALL_PREFIX=/usr \
         -D CMAKE_POSITION_INDEPENDENT_CODE=ON \
+        -D CMAKE_RANLIB=/usr/bin/llvm-ranlib \
         -D CMAKE_SKIP_RPATH=ON \
+        -D LLVM_ENABLE_LTO=Thin \
         -D LLVM_EXTERNAL_LIT=/usr/bin/lit \
         -D LLVM_EXTERNAL_SPIRV_HEADERS_SOURCE_DIR=/usr/include/spirv/ \
         -D LLVM_SPIRV_INCLUDE_TESTS=ON \
+        -D LLVM_USE_LINKER=lld \
         -D BUILD_SHARED_LIBS=ON \
         -Wno-dev
 
@@ -50,6 +65,8 @@ check() {
 }
 
 package() {
+    # move llvm-libs-minimal-git dependency here
+    depends+=('llvm-libs-minimal-git')
 
     make -C _build DESTDIR="${pkgdir}" install
     install -Dm755 _build/tools/llvm-spirv/llvm-spirv -t "$pkgdir"/usr/bin
