diff --git a/PKGBUILD b/PKGBUILD
index 0b9b86f..0171ead 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,14 +2,13 @@
 # Contributor: Eric Engestrom <aur [at] engestrom [dot] ch>
 
 pkgname=spirv-tools-git
-epoch=3
-pkgver="2023.5".3845.2a238ed24
+pkgver=2023.6.rc1.r13.gde3d5acc0
 pkgrel=1
 pkgdesc='API and commands for processing SPIR-V modules'
 url='https://github.com/KhronosGroup/SPIRV-Tools'
 arch=('i686' 'x86_64')
-license=('custom')
-groups=('vulkan-devel')
+license=('Apache-2.0')
+groups=('vulkan-devel' 'mesa-minimal')
 source=('git+https://github.com/KhronosGroup/SPIRV-Tools'
               'git+https://github.com/google/googletest.git'
               'git+https://github.com/google/effcee.git'
@@ -49,26 +48,16 @@ prepare() {
   ln -s "$srcdir"/effcee
   ln -s "$srcdir"/re2
   popd
-  
-#  patch --directory="SPIRV-Tools" --forward --strip=1 --input="${srcdir}"/212dd9247ad60d5dcd262e43c24ec92de1a8ef1d.patch
-#  patch --directory="SPIRV-Tools" --forward --strip=1 --input="${srcdir}"/5cf47a9f90e192a1c799a04d7416b6dcd7dd59a8.patch
-  
-  cmake   -S SPIRV-Tools -B _build "${cmake_args[@]}" -Wno-dev
-  make -C _build spirv-tools-build-version
 }
 
 pkgver() {
-  local _ver1 _ver2
-  # read fails if only 1 var is used
-  IFS="," read -r _ver1 _ver2 < _build/build-version.inc || [ -n "_ver1" ]
-  
-  cd SPIRV-Tools  
-  echo ${_ver1//v/}.$(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+  git -C SPIRV-Tools describe --long --tags --exclude='*sdk-*' | sed 's/v//;s/\([^-]*-g\)/r\1/;s/-/./g'
 }
 
 
 
 build() {
+  cmake -S SPIRV-Tools -B _build "${cmake_args[@]}" -Wno-dev
   make -C _build
 }
 
