diff --git a/PKGBUILD b/PKGBUILD
index 787dc94..edad2ca 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,7 +9,7 @@ pkgdesc='API and commands for processing SPIR-V modules'
 url='https://github.com/KhronosGroup/SPIRV-Tools'
 arch=('i686' 'x86_64')
 license=('custom')
-groups=('vulkan-devel')
+groups=('vulkan-devel' 'mesa-minimal')
 source=('git+https://github.com/KhronosGroup/SPIRV-Tools'
               'git+https://github.com/google/googletest.git'
               'git+https://github.com/google/effcee.git'
@@ -64,7 +64,7 @@ pkgver() {
   _rev=$(git rev-list --count HEAD)
   _hash=$(git rev-parse --short=7 HEAD)
 
-  printf "$_ver1"".r""$_rev"".""$_hash"
+  printf '%s.r%s.%s' "${_ver1}" "${_rev}" "${_hash}"
 }
 
 
