diff --git a/PKGBUILD b/PKGBUILD
index 10892c2..26ec6f4 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,15 +4,36 @@ pkgname=svp-bin
 pkgver=4.5.210
 _pkgver=${pkgver}-2
 pkgrel=6
-epoch=
 pkgdesc="SmoothVideo Project 4 (SVP4)"
 arch=('x86_64')
 url="https://www.svp-team.com/wiki/SVP:Linux"
-license=('LicenseRef-custom')
-groups=()
-depends=(libmediainfo libusb lsof vapoursynth qt5-svg qt5-script qt5-declarative xdg-utils)
+license=(
+  'BSD-3-Clause'
+  'GPL-2.0-only'
+  'GPL-2.0-or-later'
+  'LGPL-3.0-only'
+  'LGPL-3.0-or-later'
+  'MIT'
+  'Unlicense'
+  'LicenseRef-SVP-Team'
+  'LicenseRef-scancode-hidapi'
+  'LicenseRef-scancode-public-domain'
+)
+depends=(
+  'bash'
+  'gcc-libs'
+  'glibc'
+  'libmediainfo'
+  'libusb' 'libusb-1.0.so'
+  'lsof'
+  'python'
+  'qt5-declarative'
+  'qt5-script'
+  'qt5-svg'
+  'xdg-utils'
+  'vapoursynth'
+)
 makedepends=(p7zip)
-checkdepends=()
 # Youtube-dl is bundled with SVP in extensions directory
 optdepends=(
 	'mpv: Video player'
@@ -23,13 +44,9 @@ optdepends=(
 	'vlc: Video player'
 )
 provides=('svp')
-conflicts=()
-replaces=()
-backup=()
+conflicts=('svp')
 options=(!strip)
 install=${pkgname}.install
-changelog=
-#source=("https://gist.githubusercontent.com/phiresky/1e2cbd30bed4e5978771af232d11afd1/raw/svp4-linux.$pkgver.tar.bz2")
 source=("https://www.svp-team.com/files/svp4-linux.${_pkgver}.tar.bz2")
 # I am rehosting the binaries taken from
 # http://www.svp-team.com/files/svp4-linux-64.tbz2
@@ -37,9 +54,7 @@ source=("https://www.svp-team.com/files/svp4-linux.${_pkgver}.tar.bz2")
 # so they are correctly versioned and old versions still exist
 # update 2019-10-06: svp-team.com now uses versioned file names.
 # I'll keep rehosting them for now since i don't trust them to actually keep old versions.
-noextract=()
 sha256sums=('758f6e43d8f34c788dd974a73ab5ed1c3d75208256eadf415331864057d9fb28')
-validpgpkeys=()
 
 prepare() {
 	rm -rf "${srcdir}/installer"
@@ -62,7 +77,7 @@ prepare() {
 #}
 
 package() {
-	mkdir -p "${pkgdir}"/{opt/svp,usr/bin,usr/share/licenses/${pkgname}}
+	mkdir -p "${pkgdir}"/{opt/svp,usr/bin,usr/share/licenses}
 	if [[ -d "${srcdir}/extracted/licenses" ]]; then
 		mv "${srcdir}/extracted/licenses" "${pkgdir}/usr/share/licenses/${pkgname}"
 	fi
