diff --git a/PKGBUILD b/PKGBUILD
index 9cbcc26..7365060 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,7 +8,7 @@ pkgdesc="Official Tutanota email client"
 arch=('x86_64')
 url="https://tuta.com"
 license=('GPL3')
-depends=('alsa-lib' 'gtk3' 'libsecret' 'nss')
+depends=('alsa-lib' 'dbus' 'gtk3' 'hicolor-icon-theme' 'libsecret' 'libxss' 'nss')
 makedepends=('git' 'nvm' 'python')
 source=("https://github.com/tutao/tutanota/archive/$pkgname-release-$pkgver.tar.gz"
         "$pkgname.desktop")
