diff --git a/PKGBUILD b/PKGBUILD
index b5ca206..9e71a04 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,7 +8,18 @@ pkgdesc="Official Tutanota email client"
 arch=('x86_64')
 url="https://tuta.com"
 license=('GPL-3.0-or-later')
-depends=('alsa-lib' 'gtk3' 'libsecret' 'nss')
+depends=(
+  'alsa-lib'
+  'dbus' 'libdbus-1.so'
+  'expat' 'libexpat.so'
+  'gcc-libs'
+  'glib2' 'libgio-2.0.so' 'libglib-2.0.so' 'libgobject-2.0.so'
+  'glibc'
+  'gtk3'
+  'hicolor-icon-theme'
+  'libsecret'
+  'libxss'
+  'nss')
 makedepends=('git' 'nvm' 'python')
 source=("https://github.com/tutao/tutanota/archive/$pkgname-release-$pkgver.tar.gz"
         "$pkgname.desktop")
