diff --git a/PKGBUILD b/PKGBUILD
index c07dcf1..61649a3 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -5,16 +5,16 @@
 
 pkgbase=vhba-module
 pkgname=(
-  vhba-module
   vhba-module-dkms
 )
 pkgver=20240202
-pkgrel=8
+pkgrel=1
+epoch=1
 pkgdesc="Kernel module that emulates SCSI devices"
 url="https://cdemu.sourceforge.io/"
 arch=(x86_64)
 license=(GPL-2.0-or-later)
-makedepends=(linux-headers)
+#makedepends=(linux-headers)
 source=(
   "https://downloads.sourceforge.net/cdemu/$pkgbase-$pkgver.tar.xz"
   dkms.conf
@@ -34,7 +34,7 @@ prepare() {
 
 build() {
   cd $pkgbase-$pkgver
-  make KERNELRELEASE="$(</usr/src/linux/version)"
+  #make KERNELRELEASE="$(</usr/src/linux/version)"
 }
 
 package_vhba-module() {
