diff --git a/PKGBUILD b/PKGBUILD
index c7102e2..8a1f725 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,10 +8,11 @@ _vlcver=3.0.20
 # optional fixup version including hyphen
 _vlcfixupver=
 pkgver=${_vlcver}${_vlcfixupver//-/.r}
-pkgrel=4
+pkgrel=4.1
 pkgdesc='Multi-platform MPEG, VCD/DVD, and DivX player'
 url='https://www.videolan.org/vlc/'
 arch=('x86_64')
+groups=('extra-rebuild')
 license=(
   'GPL-2.0-or-later'
   'LGPL-2.1-or-later'
