diff --git a/PKGBUILD b/PKGBUILD
index e847be9..2a2d546 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -12,6 +12,7 @@ pkgrel=8
 pkgdesc='Multi-platform MPEG, VCD/DVD, and DivX player'
 url='https://www.videolan.org/vlc/'
 arch=('x86_64')
+groups=('extra-rebuild')
 license=(
   'GPL-2.0-or-later'
   'LGPL-2.1-or-later'
@@ -230,8 +231,10 @@ prepare() {
 build() {
   cd ${pkgname}-${_vlcver}
 
+  export CFLAGS="${CFLAGS/FORTIFY_SOURCE=3/FORTIFY_SOURCE=2}"
   export CFLAGS+=" -I/usr/include/samba-4.0 -ffat-lto-objects"
   export CPPFLAGS+=" -I/usr/include/samba-4.0"
+  export CXXFLAGS="${CXXFLAGS/FORTIFY_SOURCE=3/FORTIFY_SOURCE=2}"
   export CXXFLAGS+=" -std=c++17"
   export PKG_CONFIG_PATH="/usr/lib/ffmpeg4.4/pkgconfig"
   export LUAC=/usr/bin/luac
@@ -356,3 +359,4 @@ package() {
 }
 
 # vim: ts=2 sw=2 et:
+pkgrel=${pkgrel}.1
