diff --git a/PKGBUILD b/PKGBUILD
index 6a32c89..c034545 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,7 +7,7 @@ gitname=sct
 pkgdesc="Improved sct (set screen color temperature)"
 arch=('i686' 'x86_64')
 url="https://github.com/faf0/${gitname}"
-license=('Public Domain')
+license=('custom:Public Domain')
 depends=()
 makedepends=('git' 'libx11' 'libxrandr')
 source=("https://github.com/faf0/sct/archive/refs/tags/${pkgver}.tar.gz")
